---
title: "Translation on Demand as a post-transcriptional regulation mechanism of ES cell differentiation"
subtitle: "Master thesis for acquisition of the degree of Master of Science (M.Sc.) in Computational Biology of the faculty of mathematics and natural sciences at the University of Cologne"
author: "submitted by Elias Schwall"
date: "today"
date-format: long
fontsize: 12pt
mainfont: "Helvetica"
geometry: margin=2.5cm
number-sections: true
header-includes:
- \usepackage{authblk}
- \usepackage[fontsize=12pt]{scrextend}
- \usepackage{setspace}
- \setstretch{1.5}
- \usepackage{graphicx}
- \pagenumbering{gobble}
- \usepackage{fancyhdr}
- \pagestyle{fancy}
- \fancyhf{}
- \fancyfoot[R]{\thepage}
format: 
  pdf:
    documentclass: article
    classoption: "12pt"
    link-citations: true
    crossref:
      custom:
        - kind: float
          key: suppfig
          latex-env: suppfig
          reference-prefix: Figure S
          space-before-numbering: false
          latex-list-of-description: Supplementary Figure
        - kind: float
          key: supptbl
          latex-env: supptbl
          reference-prefix: Table S
          space-before-numbering: false
          latex-list-of-description: Supplementary Table
          caption-location: top
bibliography: Master_Thesis.bib
---

```{=latex}
\textbf{Reviewers}: \newline
Prof. Dr. Andreas Beyer \newline
Prof. Dr. Achim Tresch

\textbf{Supervised by}: \newline
Luke Brandwood

\vspace*{\fill}
\vspace*{-2cm}  
\begin{figure}[h]
    \centering
    \begin{minipage}{0.5\textwidth}
        \centering
        \includegraphics[width=\textwidth]{CECAD_logo.jpg}
    \end{minipage}\hfill
    \begin{minipage}{0.4\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Uni_Köln_Logo.png}
    \end{minipage}
\end{figure}

\vspace*{\fill}

\newpage

\pagenumbering{Roman}

\section*{Declaration of originality}

Hiermit versichere ich an Eides Statt, dass ich die vorliegende Arbeit selbstständig
und ohne die Benutzung anderer als der angegebenen Hilfsmittel angefertigt habe. Alle
Stellen, die wörtlich oder sinngemäß aus veröffentlichten und nicht veröffentlichten
Schriften entnommen wurden, sind als solche kenntlich gemacht. Die Arbeit ist in gleicher
oder ähnlicher Form oder auszugsweise im Rahmen einer anderen Prüfung noch nicht
vorgelegt worden. Ich versichere, dass die eingereichte elektronische Fassung der
eingereichten Druckfassung vollständig entspricht. \newline \newline
I hereby declare that this thesis is my own work and that I have not used any
sources other than those listed in the bibliography. Content from published or unpublished
worksthat has been quoted directly or indirectly or paraphrased isindicated as such. The thesis has not been submitted in the same or similar form or in part for any other academic award. The electronic version I have submitted is completely identical to the hard copy version submitted.

\vspace*{\fill}

Cologne, \today \hspace*{\fill} Elias Schwall

\newpage

\section*{Acknowledgement}

This thesis would not have been possible without the support and encouragement of numerous individuals. I would like to express my deepest gratitude to reviewers Prof. Dr. Andreas Beyer and Prof. Dr. Achim Tresch for granting me the opportunity to undertake my master's thesis at the Exzellenzcluster for Cellular Stress Responses in Aging-Associated Diseases (CECAD). Prof. Dr. Beyer consistently provided exceptional insights into grappling with scientific questions as well as questioning scientific assumptions and offered invaluable feedback. He always was avaible for addressing my problems. \newline I would also like to extend my heartfelt gratitude to Luke Brandwood, who supervised my project. He introduced me to the field of stem cell research and Translation on Demand by teaching me day after day during my time at the AG Beyer. Every day I could discuss new insights and strategies with him, and his mentoring and feedback were paramount for progressing in this project. I was extremely grateful for having had the chance to support him in his PhD project. My time at the AG Beyer was extremely fun not only because of learning new things everyday but also because of the people working there. Therefore, I would also like to thank all other members of AG Beyer. \newline I would like to offer a special note of thanks to my friends and mentors from Aimed Analytics, Dr. Kevin Baßler and Dr. Patrick Günther. They were very considerate during the time of my master's thesis and always offered me support and feedback. \newline The experience of my master's study program was significantly enriched by my friend and fellow student Amir Hyseni. Working alongside him towards the same goal made it all the more fun. \newline Lastly, I would be remiss if I didn’t acknowledge the enormous support from my family, which was indispensable for the completion of this thesis. I extend my sincerest thanks to my father Robert and my mother Maria-Theresia, who deserve special recognition for supporting in every way and always being my biggest cheerleaders. My older brother Johannes, whose presence and readiness to help have been a constant in my life, and my partner Patricia, who constantly supported and encouraged me throughout this intensive period of work.


\newpage

\tableofcontents

\newpage

\setcounter{page}{1}
\pagenumbering{arabic}
```

# Abstract

This study explores Translation on Demand (ToD) as a post-transcriptional regulatory mechanism crucial for the differentiation of embryonic stem (ES) cells. Focusing on mouse embryonic stem cells (mESCs), the study examines how ToD facilitates the transition from a naive pluripotent state to a primed state, a key step in early development. By integrating RNA sequencing and quantitative proteomics data, our research identifies specific genes where protein levels increase rapidly without corresponding changes in mRNA levels, indicating ToD regulation. The methodology involves a multi-omics approach to analyze the dynamics of ToD, revealing its role in modulating the proteomic landscape during stem cell differentiation. We show that ToD candidates are enriched in pathways critical for development, such as Notch and Wnt/β-catenin signaling. Next, we demonstrate that transcription factors (TFs) are particularly susceptible to ToD regulation. Our analysis reveals that TFs identified as potential ToD candidates exhibit the highest activity levels precisely during their ToD regulation phase. Additionally, we invastigate the involvement of RNA-binding proteins (RBPs) and untranslated regions (UTRs) in enabling ToD, providing insights into the molecular mechanisms that facilitate this regulatory mechansim. Our results demonstrates that ToD significantly influences early development by allowing rapid protein synthesis in response to developmental signals, thereby supporting the shift from pluripotency to lineage specification. This work enhances the understanding of post-transcriptional regulation in stem cell biology and provides a framework for identifying ToD candidates across different biological contexts, potentially informing strategies for regenerative medicine and developmental biology.

\newpage

# Introduction

## Mouse embryonic stem cells

Mouse embryonic stem cells (mESCs) are defined by their self-renewal capabilities and pluripotency, allowing them to differentiate into any cell type of the three germ layers. While this differentiation process is limited in vivo, pluripotent stem cells (PSCs) cultivated in vitro can self-renew indefinitely under specific culture conditions [@nicholsNaivePrimedPluripotent2009]. One such condition is the 2i+LIF (two inhibitors with leukemia inhibitory factor) medium, which maintains the naive pluripotent state of ESCs, closely mimicking the inner cell mass of pre-implantation mouse embryos. \newline The continuum of pluripotency is described by a progression from the naive state to a primed state, characterized by the priming for differentiation [@nicholsNaivePrimedPluripotent2009]. Naive mESCs serve as a model for the inner cell mass of the pre-implantation embryo, exhibiting minimal lineage commitment. In contrast, epiblast stem cells (EpiSCs), which are isolated from later developmental stages, correspond to the post-implantation epiblast and are primed for differentiation [@boroviakAbilityInnercellmassCells2014a]. \newline During the transition from the naive to the primed states, sequential changes occur at multiple molecular levels, encompassing alterations in signaling pathways, epigenetic markers, and gene expression [@dejosezPluripotencyNuclearReprogramming2012a]. Specifically, early modifications in the phosphoproteome serve as indicators of these state transitions, initiating a cascade of changes that ultimately impact the transcriptome and proteome [@needhamIlluminatingDarkPhosphoproteome2019a]. This progression underscores the dynamic nature of pluripotency and highlights the intricate molecular mechanisms governing cell fate decisions in early development. \newline It is worth noting that early mESCs development differs from human embryonic stem cells (hESCs). While the molecular mechanisms controlling developmental processes are evolutionarily conserved, the speed at which these processes occur can vary significantly between species. For instance, the differentiation of motor neurons from embryonic stem cells is more than twice as fast in mice compared to humans [@rayonSpeciesspecificDevelopmentalTiming2020]. \newline Regardless of the species, however, early developing embryonic stem cells need to undergo rapid changes in their gene expression profiles to ensure proper development and differentiation.

## 2i Condition for maintaining naive pluripotency

Since serum contains undefined factors that can induce differentiation, the 2i medium was developed to provide a defined, chemically controlled environment that preserves mESCs in the naive pluripotent state [@yingGroundStateEmbryonic2008]. The 2i medium relies on the synergy between two inhibitors, PD0325901 and CHIR99021, to sustain mESCs in an undifferentiated state. PD0325901, a MEK inhibitor, blocks the MEK/ERK signaling pathway, suppressing differentiation signals from FGF and ERK that would otherwise prompt mESCs to exit pluripotency (see @fig-2i). Meanwhile, CHIR99021, a GSK3α/β inhibitor, stabilizes β-catenin which activates Wnt/β-catenin signaling, promoting the expression of self-renewal genes and reinforcing pluripotency [@yingGroundStateEmbryonic2008]. Together, these inhibitors create a controlled environment that preserves mESCs in a ground state resembling the undifferentiated pre-implantation epiblast [@yingGroundStateEmbryonic2008]. Leukemia inhibitory factor (LIF) activates the JAK–STAT3 pathway to suppress differentiation. When combined with the two inhibitors of the 2i medium, this signaling network maintains mESCs in a naïve pluripotent state, preserving their full, unbiased differentiation potential. [@yingGroundStateEmbryonic2008]. \newline The removal of the 2i medium triggers the transition from the naive pluripotent state by reactivating the MEK/ERK and GSK3 pathways, allowing ERK signaling to promote differentiation and reducing Wnt signaling to enable lineage specification. This shift initiates a cascade of biological changes, including the downregulation of naive markers (such as Nanog and Esrrb) and upregulation of formative markers (such as Otx2), along with epigenetic remodeling, altered chromatin accessibility, and a metabolic shift from glycolysis to oxidative phosphorylation (see @fig-2i) [@wrayInhibitionGlycogenSynthase2011a].

![**2i mechanism of stabilizing naive pluripotency** \newline The inhibition of Mek/Erk through 2i-dependent mechanisms prevents the phosphorylation of Erf and Cic, resulting in the stabilization of these factors within the nucleus. These transcriptional effectors may play a role in activating genes associated with naïve pluripotency while repressing those involved in primed/formative states. Cdk8/19i influences the phosphorylation of proteins that are part of the elongation and transcription machinery, including the super elongation complex (SEC), RNA polymerase II (RNPII), and Mediator (MED). This could lead to the rapid activation of genes regulated by super-enhancers, which include factors that stabilize the naïve state as well as certain factors from the primed/formative states that are also upregulated in response to Cdk8/19i. The activation of the naïve circuitry through 2i and Cdk8/19i may support the enhanced mitochondrial oxidative phosphorylation (OXPHOS) capacity of these cells. The degradation of Lin28a might alleviate its repressive effects on target mRNAs, leading to the upregulation of mitochondrial proteins among others. The balance between autophagy and protein synthesis may sustain the translation of these mitochondrial components. Additionally, the re-wiring of mitochondrial sources for one-carbon units may further support the translation of mitochondrial-encoded proteins. Furthermore, the increased availability of S-adenosylmethionine (SAM) levels, due to NNMT upregulation, combined with the impairment of DNA methylation caused by Uhrf1 degradation, contributes to the characteristic hypomethylated DNA state typical of 2i conditions [@martinez-valDissectionTwoRoutes2021].](figures/2i_Figure.png){#fig-2i fig-pos="H"}

## Translation on Demand

During this transitions, embryonic stem cells (ESCs) need to undergo drastic changes in protein expression to adapt and develop efficiently. There must be mechanisms, these cells undergoing fundamental changes in their transcriptome and proteome can employ, that allow for the rapid development. These mechanisms might also be one of the reasons why certain species exhibit a faster rate of development. One of these mechanisms might be Translation on Demand (ToD [@brockmannPosttranscriptionalExpressionRegulation2007]). Mechanisms such as ToD, can allow for swift adjustments in protein levels to aid these rapid changes. This ensures that cells can quickly respond to developmental cues and environmental changes, maintaining the necessary pace of development. Such mechanisms are crucial for adapting to the specific temporal requirements, ensuring that ESCs can effectively progress through the stages of development. \newline We define ToD as an increase in protein abundance due to enhanced translation, occurring without any changes in mRNA levels of the corresponding gene. In other words, ToD is a regulatory mechanism where translation efficiency is selectively increased for transcripts of specific genes. This upregulation allows for rapid adjustments in protein levels, supporting cellular responses that need immediate protein synthesis without requiring new mRNA transcription. Such translation driven increases could be particularly useful in situations where quick adaptations are necessary, as it enables cells to respond to changes in the environment or developmental cues by rapidly elevating the protein output from existing mRNA pools. \newline In vivo, there are already described mechanisms that facilitate forms of selective, rapid translation of mRNAs. For example, neurons need local translation at active synapses for quick cellular responses [@holtCentralDogmaDecentralized2013]. In early embryos, maternal mRNA is stored and activated after fertilization, enabling the embryo to rapidly synthesize proteins necessary for early development without new mRNA synthesis [@deleonHalflivesRelativeAmounts1983]. In stress response, rapid and selective synthesis of stress proteins, such as heat shock proteins (HSPs), is facilitated by ToD, allowing cells to quickly adapt and protect themselves under stress conditions without increasing mRNA levels [@holcikTranslationalControlStress2005]. ToD has also been shown in the Yeast Saccharomyces cerevisiae as a post-transcriptional expression regulator [@beyerPosttranscriptionalExpressionRegulation2004]. \newline We know that cells use ToD as a tool to selectively and rapidly change their proteomic landscape. However, the mechanism by which cells facilitate ToD remains elusive. In the following sections, we explore potential mechanisms, whether specific or in combination, that might enable cells to facilitate ToD.

## ERK and mTOR signalling pathway and its effects on translation

ERK signaling influences translation primarily by modulating the activity of eukaryotic translation initiation factor 4E (eIF4E) [@lavoieERKSignallingMaster2020]. ERK activates MAPK-Interacting Kinases (MNK)1 and MNK2 kinases, which phosphorylate eIF4E, enhancing its binding to the mRNA cap structure and promoting the formation of the eIF4F complex (see @fig-erg_and_mTOR). This complex, consisting of eIF4E, eIF4G, and eIF4A, is essential for recruiting the ribosome to mRNA, scanning for the start codon, and initiating protein synthesis. ERK signaling can selectively enhance the translation of specific mRNAs, particularly those with complex 5′ UTRs, leading to the production of proteins involved in cell growth and survival. Additionally, ERK interacts with other pathways, such as mTOR, to coordinate translation regulation in response to various signals, thus optimizing cellular responses to environmental changes. \newline The mTOR pathway complements the role of ERK in translation regulation. It comprises two distinct complexes: mTORC1 and mTORC2. mTORC1 is primarily responsible for regulating protein synthesis, autophagy, and metabolism, while mTORC2 is involved in cell survival and cytoskeletal organization. \newline Activation of mTORC1 occurs through nutrient availability, with amino acids serving as key activators, and through growth factors that activate the PI3K-AKT pathway (see @fig-erg_and_mTOR) [@maMolecularMechanismsMTORmediated2009]. mTORC1 influences translation by phosphorylating 4E-BP1, which releases eIF4E to initiate cap-dependent translation (see @fig-erg_and_mTOR) [@maMolecularMechanismsMTORmediated2009]. Furthermore, mTORC1 activates S6K1, which phosphorylates ribosomal protein S6, enhancing the translation of ribosomal proteins and elongation factors. These actions collectively increase the global protein synthesis capacity of the cell (see @fig-erg_and_mTOR) [@maMolecularMechanismsMTORmediated2009]. \newline The mTORC1 signaling pathway has a negative feedback regulation on itself and in consequence on the ERK signaling pathway through the inhibition of IRS-1 (see @fig-erg_and_mTOR). It is this negative feedback that balances translation rates to save cellular resources and prevent the accumulation of translational stress. The integration of these complex signaling pathways shows the tight regulation cells need to enforce on fundamental processes like translation to maintain homeostasis. This regulation becomes even more critical during development where ESCs need to differentiate and proliferate rapidly. During this phase, the precise control of translation rates is essential to ensure that energy and resources are optimally utilized. \newline In the naive state, mTORC signaling is low, which keeps global protein synthesis rates low. Upon the removal of 2i conditions, mTORC signaling rises, leading to an increase in global translation. This increase does not occur immediately, which suggests the presence of mTOR independent mechanisms, such as cap-independent translation. Furthermore, recent research indicates that non-canonical alternative mechanisms of cellular mRNA translation initiation, whether cap-dependent or independent, enable selective translation of mRNAs during physiological and pathological stress. These stress conditions often lead to a global downregulation of the canonical eIF4E1/cap-mediated mRNA translation and a targeted reprogramming of the cellular proteome, as seen in tumor development and malignant progression [@maheNoncanonicalMRNATranslation2024]. Cap-independent translation and other translational regulation mechanisms are tightly controlled by the make up of untranslated regions (UTRs) of mRNAs [@meyer5UTRM6A2015].

![**ERK and mTORs effects on eIF4E regulation** \newline The ability of eIF4E to bind to the mRNA cap makes it a key regulator of translation. A crucial step in translation initiation involves eIF4E binding to the mRNA cap, facilitating the formation of the eIF4F complex on the cap structure (comprising a 7mGp linked to the first nucleotide). The eIF4F complex includes eIF4E, eIF4G (a scaffolding protein), and eIF4A (a helicase). The successful assembly of the eIF4F complex on the mRNA cap promotes the recruitment of the pre-initiation complex (PIC), followed by scanning of the 5′ UTR to locate the start codon AUG and the joining of the 60S ribosomal subunit, marking the completion of translation initiation. eIF4E acts as a downstream effector of both the mTORC1 pathway (via 4E-BP-dependent repression) and the ERK pathway (via eIF4E phosphorylation by MNK 1/2). The activities of mTORC1 and ERK signaling pathways are influenced by various external signals and internal cues [@uttamEIF4EDependentTranslationalControl2018].](figures/ERK_and_mTOR_effect_on_translation.jpg){#fig-erg_and_mTOR fig-pos="H"}

## Untranslated Regions in mRNAs

mRNA is composed of coding sequences, containing exons and introns, interspersed by non coding segments known as UTRs. Despite not encoding any information for protein production itself, UTRs are pivotal for regulating gene expression post‐transcriptionally. There are two types of UTRs. The 5' UTR located immediately upstream of the start codon, while the 3' UTR is positioned immediately downstream of the coding region. Together they integrate diverse cis‐regulatory elements that dictate mRNA stability, localization, translation initiation, and subsequently protein functionality. UTRs can vary considerably in length and composition, with longer 3' UTRs often correlating with higher organismal complexity and providing more sites for regulatory factor binding. \newline Through secondary structures, upstream open reading frames (uORFs), internal ribosome entry sites (IRESs), microRNA (miRNA) binding sites, and binding sites for RNA‐binding proteins (RBPs), UTRs provide cells with a modular toolbox to regulate gene expression in response to developmental cues, stress conditions, and disease processes [@mayrWhatAre32019; @leppekFunctional5UTR2018]. \newline The 3' UTR is rich in regulatory motifs that can influence mRNA stability and turn over. AU-rich elements (AREs) for example and other decay signals within the 3' UTR can either enhance or reduce mRNA half-life through interactions with specific RBPs, thereby indirectly controlling protein abundance [@shawConservedAUSequence1986]. 3' UTRs also function as critical determinants for mRNA localization within the cell by containing sequences that direct mRNAs to specific subcellular compartments [@meltonTranslocationLocalizedMaternal1987]. \newline The 5' UTR frequently recruits RBPs and miRNAs that modulate translation efficiency by either stabilizing or repressing ribosome engagement. The formation of RNA secondary structures within the 5' UTRs, such as hairpins and G-quadruplexes, can impede ribosome scanning and modulate translation initiation [@leppekFunctional5UTR2018]. These structures can again either enhance or inhibit translation depending on their stability, location relative to the cap, and interactions with RNA helicases that resolve inhibitory folding.

## RNA binding proteins and miRNAs

As stated earlier, RBPs and miRNAs are essential regulators for mRNA turnover as well as translational efficiency. RBPs constitute a large and diverse class of proteins that govern nearly every step of RNA metabolism, from splicing and transport to stability and translation [@corleyHowRNABindingProteins2020]. RBPs bind mRNA through modular RNA-binding domains (RBDs), which primary sequence of amino acids dictates the specificity of the RBP [@lundeRNAbindingProteinsModular2007], as well as intrinsically disordered regions (IDRs) to allow dynamic interactions with numerous RNA substrates. Besides influencing the turnover rate of specific mRNAs, RBPs also regulate mRNA stability. They form complexes with other proteins to recruit for example ribonucleoproteins (RNPs) from the translation machinery, ensuring proper RNA folding, translocation and translational regulation [@glisovicRNAbindingProteinsPosttranscriptional2008]. Another crucial way that RBPs regulate mRNA stability and translation efficiency is through the formation of closed-loop structures. This process relies on the interaction between factors attached to the 5' cap and the 3' poly(A) tail of mRNAs, which enhances translation efficiency by physically circularizing the mRNA. RBPs contribute to this mechanism by binding simultaneously to both the cap and tail regions or by recruiting bridging factors like PABP [@zhaoRevisitingPolyAbindingProteins2021]. This increases ribosome recycling and protects mRNAs from degradation enzymes. \newline Similar to RBPs another prominent example of post-transcriptional regulation through direct interaction with mRNAs are miRNAs. miRNAs are a ubiquitous class of short, non‐coding RNA molecules. These about 21–24 nucleotide long transcripts are generated through a tightly controlled multistep biogenesis process and exert profound influence on the translation of mRNAs. miRNAs influence translation by primarily binding to partially complementary sequences in the 3' UTRs of target mRNAs, although binding sites can also reside in the coding region or 5' UTR in some cases [@filipowiczMechanismsPosttranscriptionalRegulation2008]. The binding is largely governed by the “seed” region, comprising nucleotides 2–8 at the miRNA’s 5' end, which is critical for target specificity [@jacksonHowMicroRNAsRegulate2007]. Once bound, the miRNA-RISC complex exerts its regulatory effects via several overlapping mechanisms. One major mechanism involves the inhibition of translation initiation. The miRNA-RISC can interfere with cap recognition by eIF4E or disrupt the assembly of the initiation complex, thereby preventing the recruitment of ribosomal subunits to the target mRNA [@pillaiRepressionProteinSynthesis2007]. This blockade of initiation results in a halt of protein synthesis while often leaving the mRNA intact in a translationally silent state [@wilczynskaComplexityMiRNAmediatedRepression2015]. In mESCs specifically, families of miRNAs have been shown to maintain pluripotency and suppressing genes that drive differentiation [@meltonOpposingMicroRNAFamilies2010]. The fact that, ERK signalling globally downregulates miRNAs, further highlights the role of miRNAs to maintain pluripotency and ERKs to exit the naive state [@sunERKActivationGlobally2016].

## Epigenetic and Epitranscriptomic regulation of gene expression

Lastly epigenetic regulation, although indirectly, can influence translation. Classical epigenetic marks such as DNA methylation and histone modifications determine transcription start site selection and alternative exon usage, thereby dictating the 5' UTR features and coding potential of mRNAs, which in turn influence their translational competence and the diversity of protein isoforms produced [@leenenDNAMethylationConducting2016]. In mESCs specifically, the incorporation of specific histone variants, such as H3.3, has been shown to modulate the chromatin environment. Regulatory complexes can modify chromatin to control gene expression during development by influencing the accessibility and stability of specific genomic regions [@banaszynskiHiradependentHistoneH332013]. \newline Additionally, post‐transcriptional chemical modifications of RNA (the epitranscriptome) represent a dynamic and reversible system that regulates mRNA stability, translation initiation, elongation speed, and termination processes. Similar to epigenetic modifications, key modifications such as N6‐methyladenosine (m6A), N1‐methyladenosine (m1A), pseudouridine (Ψ), and 5‐methylcytosine (m5C) are installed by dedicated writer enzymes, recognized by specific reader proteins and removed by erasers, collectively mediating changes in RNA structure and ribosome engagement [@peerEpitranscriptomeTranslationRegulation2019]. \newline Collectively, these mechanisms allow cells to continue efficiently managing protein synthesis in response to various internal and external stimuli. By finely tuning the translation process, cells can adapt to changing conditions, conserve resources, and ensure that proteins are synthesized correctly and at the right time. This regulation is crucial for maintaining cellular homeostasis and supporting functions such as growth, differentiation, and response to stress. In this project, we propose another mechanism by which cells selectively and rapidly increase the translational efficiency of certain transcripts. ToD might be enabled by one or multiple of the stated mechanisms by which cells regulate translation.

## Aim of this study

In this study, we set out to characterize ToD as a post-transcriptional regulator mechanism of mESC differentiation by addressing three core questions:

1.  Which genes are subject to ToD in early development?

We integrate RNA-seq and quantitative proteomics time-course data to identify transcripts whose protein outputs change more rapidly than their mRNA levels [@yangMultiomicProfilingReveals2019].

2.  Why are these genes regulated by ToD?

Alongside finding genes that are subject to ToD regulation, we want to illustrate why these specific genes are prioritized and to uncover the underlying mechanisms of this regulation. We analyze the functional roles and pathway enrichments of the candidate set to understand why rapid translational tuning is advantageous during early differentiation. Additionally, we are interested in understanding the gene regulatory network and proteome changes that occur as mESCs exit pluripotency, as these regulatory networks are most likely to be regulated by ToD.

3.  Which mechanisms enable ToD regulation?

Furthermore, we want to understand which posttranscriptional regulation mechanism are essential for enabling mESCs to facilitate ToD. We interrogate each candidate’s mRNA structural features 5' and 3' UTR sequences, upstream open reading frames, and known RBP/miRNA binding sites to infer cis-regulatory mechanisms. By examining high-resolution differentiation time-courses, we pinpoint the developmental window during which translational shifts take place. \newline By analyzing the temporal dynamics within these data layers, we can distinguish cases where protein levels increase independently of mRNA changes, a hallmark of ToD regulation. This approach provides a comprehensive view of the post-transcriptional mechanisms that contribute to protein abundance, offering insights into gene-specific regulatory processes. Understanding the more nuanced relationship between RNA and protein expression, as well as the timing of regulation, will provide valuable insights into the fundamental principles of molecular biology and enhance our comprehension of cellular regulatory networks as well as early development. \newline Furthermore, this study should serve as a framework to detect ToD candidates in by leveraging the use of transcriptome and proteome data together. We also aim to validate our findings and approaches with two other datasets to check the validity of our methods and see if we obtain similar results. This validation will help ensure that the mechanisms identified are robust and applicable across different biological contexts. Together, these analyses will provide a comprehensive “who, why, how, and when” picture of Translation on Demand in mESC fate decisions.

# Methods and tools

## Experimental Approach and Workflow

To research ToD, we exploit the fact that protein translation is largely inhibited under 2i conditions. By sampling mouse embryonic stem cells under 2i conditions and then periodically after 2i removal, we can take snapshots of the early changes in the transcriptome and proteome (see @fig-workflow). Upon 2i removal, we trigger the transition from the naive pluripotent state to the primed formative state. This allows us to detect changes in the translational behavior of genes aiding this transition. \newline We connect transcriptomic and proteomic dynamics by comparing their Log Fold Change (LFC) between time points. This way, we can compare qualitative trends for transcript and protein abundance rather than comparing them by count values. After identifying genes that are likely ToD-regulated, we assess their biological function and place them in the context of early development (see @fig-workflow).

![**Experimental Approach and Workflow** Exemplary workflow for the Yang dataset. Proteomic and transcriptomic time courses were used to examine LFC changes at different time intervals. Filtering criteria were applied to determine if a specific gene is likely to be ToD regulated. ToD candidates were tested for functional enrichment and other annotations methods to determine their role in early development.](figures/Workflow.png){#fig-workflow fig-pos="H"}

## Data overview and preprocessing

The data for this project is comprised of three datasets with our main focus being the Yang Data, due to its higher temporal resolution in early time points and more consistent time course in the proteomic data (see @tbl-datasets). This dataset was sourced from Supplementary Table S2 (temporal proteomic data) and Supplementary Table S3 (temporal transcriptomic data) in the publication Multi-omic Profiling Reveals Dynamics of the Phased Progression of Pluripotency [@yangMultiomicProfilingReveals2019]. \newline The temporal proteomic data is in the form of Log2 Label-Free Quantification (LFQ) intensities and included the following time points: 0min, 30min, 1h, 6h, 12h, 24h, 36h, 48h, and 72h. Each time point had four biological replicates, which were also pooled. The temporal transcriptomic data consisted of non-normalized transcript counts and included the following time points: 0min, 1h, 6h, 12h, 24h, 36h, 48h, and 72h. Each time point had two biological replicates, which were averaged. Since the transcriptomic dataset did not include a 30min time point, this time point was removed from the proteomic data for further analysis (see @tbl-datasets). \newline To increase our accuracy in calling meaningful ToD candidate genes, we realigned the transciptomic data with the GRCm39 reference genome using STAR [@mudgeGENCODE2025Reference2025; @dobinSTARUltrafastUniversal2013]. \newline In the Leeb Time Course (TC) data, the temporal resolution spans from 0 to 32 hours in 4-hour intervals. For each time point, there are four biological replicates in the proteomic time course, while the transcriptomic time course does not include any replicates (see @tbl-datasets). Gaussian Process Regression (GPR) was employed to smooth out the transcriptomic trajectories as this approach leverages the more densely packed information from measurements taken at 2-hour intervals across the time course. This was a purposeful tradeoff, prioritizing a finer-grained time course over the inclusion of replicates, allowing GPR to effectively reduce noise and enhance the clarity of the temporal transcriptomic data (see @sec-GPR). \newline The Leeb Medium Change (MC) data includes time course data for 0h, 4h and 8h for the purpose of calling ToD candidates. Additionally, it includes a time course with the same temporal resolution, where 2i conditions were kept while a medium change was still applied to the cells. Each time point has four biological replicates in the proteomic time course and three replicates in the transcriptomic time course (see @tbl-datasets).

| Dataset | Temporal Resolution (h) | Replicates Transciptomics | Replicates Proteomics |
|------------------|------------------|------------------|------------------|
| Yang Data | 0, 1, 6, 12, 24, 36, 48, 72 | 2 | 4 |
| Leeb TC Data (Time Course) | 0, 4, 8, 12, 24, 32 | 1 | 4 |
| Leeb MC Data (Medium Change) | 0, 4, 8 | 3 | 4 |

: Datasets {#tbl-datasets}

## Transcriptomic Data Normalization

For the Yang dataset as well as the Leeb MC dataset we used DESeq2 to normalize the transciptomic data [@loveModeratedEstimationFold2014a]. DESeq2’s normalization procedure operates on the assumption that the majority of genes exhibit no differential expression. To normalize gene counts, DESeq2 uses the median of ratios method. This involves dividing the count of each gene in a sample by its geometric mean across all samples, resulting in a ratio (see @eq-geomean; @eq-ratio). The geometric mean serves as a measure of central tendency, representing the average gene count across samples, and provides a standard for comparing individual sample counts. The size factor for a given sample is then determined by calculating the median of these ratios (see @eq-sizefactor). This method is robust, accounting for sequencing depth and RNA composition, and effectively compensates for sample specific biases.

$$
\text{Geometric Mean}_g = \left( \prod_{s=1}^{n} \text{Count}_{gs} \right)^{1/n} 
$$ {#eq-geomean}

-   $\text{Geometric Mean}_g:\text{The geometric mean of counts for gene } g$
-   $\text{Count}_{gs}: \text{The count of gene } g \text{ in sample } s$
-   $n: \text{The total number of samples}$

$$
\text{Ratio}_{gs} = \frac{\text{Count}_{gs}}{\text{Geometric Mean}_g}
$$ {#eq-ratio}

$$\text{Size Factor}_s = \text{median}(\text{Ratio}_{gs} \mid \text{for all } g)$$ {#eq-sizefactor}

### Gaussian process regression {#sec-GPR}

The negative binomial distribution is an effective model for count data derived from both bulk and single-cell RNA sequencing. Gaussian process regression (GPR) offers a non-parametric method for modeling temporal variations in gene expression [@bintayyashNonparametricModellingTemporal2021]. GPR can be particularly useful when there are few or no replicates. By specifying a covariance function (kernel), GPR incorporates prior knowledge about the gene expression profiles. GPR naturally accounts for correlations between neighboring time points, which is beneficial in time-series data where observations are not independent. The model uses these correlations to infer missing data points and smooth out noise.

## Calculation of Fold Changes and Ratios

LFC values between consecutive time points for proteomic and transcriptomic time courses were calculated with @eq-lfcbetween.

$$
\text{logFC}_{\text{between time points}} = \text{logFC}_2(\frac{\text{Expression}_{\text{time point}_{n+1}}}{\text{Expression}_{\text{time point}_{n}}})
$$ {#eq-lfcbetween}

LFC values against the first time point (0h) for proteomic and transcriptomic time courses were calculated with @eq-lfcagainst.

$$
\text{logFC}_{\text{against time point 1}} = \text{logFC}_2(\frac{\text{Expression}_{\text{time point}_{n}}}{\text{Expression}_{\text{time point}_{1}}})
$$ {#eq-lfcagainst}

We calculated a Protein-to-RNA-LFC-Ratio (LFC-Ratio) to highlight genes with greater changes in protein than RNA, identifying possible ToD candidates (see @eq-lfcratio). A positive Protein-to-RNA FC Ratio indicates that protein expression has increased more than RNA expression, suggesting potential translational upregulation through ToD.

$$\text{LFC-Ratio} = \text{logFC}_{\text{between time points protein}} - \text{logFC}_{\text{between time points RNA}}$$ {#eq-lfcratio}

## ToD criterias {#sec-todpara}

To considere a gene potentially ToD regulated it has to pass three filtering criterias.

1.  Stable transcript expression: The LFC value of a potentially ToD regulated genes has to be within a stable range $R_s$ (e.g. -1 to 1) between two consequtive time points (see @eq-stablerange).

$$ -R_s \leq \text{LFC}_{\text{between time points}} \leq R_s$$ {#eq-stablerange}

2.  Protein abundance changes more than transcript abundance: The LFC-Ratio between protein and transcript has to be over a certain threshold (e.g. 1 would mean that the LFC value for the protein was bigger than the transcript LFC value by 1, see @eq-todthres).

$$\text{LFC-Ratio}\geq \text{ToD-threshold}$$ {#eq-todthres}

3.  Protein trajectory has to stay up: After passing the ToD-threshold the protein LFC value is only allowed to decrease by a certain amount $R_d$ (e.g 30%, see @eq-filter)

$$\text{logFC}_{\text{protein against 0h after ToD call}}\geq \text{logFC}_{\text{protein against 0h during ToD call}} \times (1-R_d)$$ {#eq-filter}

## Gene Overrepresentation Analysis with clusterProfiler {#sec-clusterProfiler}

We performed Gene Overrepresentation Analysis (ORA) on potential ToD candidates at each time point using the R package clusterProfiler [@yuClusterProfilerPackageComparing2012]. This analysis aimed to identify overrepresented biological functions, molecular activities, or cellular localizations among ToD candidates compared to a background reference. Using a hypergeometric model, clusterProfiler evaluates the enrichment of gene set terms within each set of ToD candidates, generating p-values to quantify significance. To control for false discovery rates (FDR), we applied Benjamini-Hochberg correction to the resulting p-values [@benjaminiControllingFalseDiscovery1995]. This approach highlighted time-specific biological processes potentially regulated at the translational level among ToD candidates. The corresponding complete data set (genes both present in the proteomic and transcriptomic data) was used as a background (or universe) for the analysis.

## Transciption Factor Acticity inference with decoupleR {#sec-decoupler}

We used the R package decoupleR to infere acticity of Transciption Factors (TFs) that might be subject to ToD [@badia-i-mompelDecoupleREnsembleComputational2022]. decoupleR utilizes CollecTRI, a comprehensive resource containing a curated collection of TFs and their transcriptional targets compiled from 12 different resources [@muller-dottExpandingCoverageRegulons2023]. Interactions are weighted according to their mode of regulation, either as activators or inhibitors. To infer TF activity, decoupleR uses the Univariate Linear Model (ULM) method to calculate TF enrichment scores. For each sample in our datasets and each TF in the CollecTRI network, it fits a linear model that predicts the observed gene expression based solely on the TF’s TF-Gene interaction weights. Once fitted, the obtained t-value of the slope is used as the score. A positive t-value indicates that the TF is active, while a negative t-value suggests that the TF is inactive.

## Motif enrichment with MEME {#sec-meme}

For motif enrichment in UTRs of ToD candidates, we utilized AME from the MEME suite [@mcleayMotifEnrichmentAnalysis2010; @baileyMEMESuite2015]. To use AME for our analysis purpose we used the Memes R package and the MEME CLI tool [@nystromMemesMotifAnalysis2021]. AME works by comparing a set of target sequences, in this case, the UTRs of ToD candidates, against a set of background sequences to identify motifs that are significantly enriched in the target set. When control sequences are provided, AME disregards both the FASTA scores and the sequence order. Instead, the focus is on comparing the target sequences directly against the control sequences to identify motifs that are significantly enriched in the target set using fishers exact test. Using control sequences allows the allows us to focus solely on the differences between the target and control sequences, enhancing the detection of motifs that are distinctively enriched in the target sequences.

## Software

| Software | Version | Provider | Reference |
|------------------|-------------------|------------------|------------------|
| R | 4.4.3 (2025-02-28) | R Foundation for Statistical Computing | @rcoreteamLanguageEnvironmentStatistical2025 |
| STAR | 2.7.11b | Alex Dobin | @dobinSTARUltrafastUniversal2013 |
| Docker | 27.5.1 | Docker Inc. | @merkel2014docker |
| MEME | 5.5.7 | University of Nevada, Washigton, Queensland and UCSD | @baileyMEMESuite2015 |
| BioRender | 2025 | BioRender | @biorenderScientificImageIllustration2025 |

: Software used for the analysis and visualizations {#tbl-software}

### R packages

| Package Name    | Version | Reference                                         |
|------------------------|------------------------|------------------------|
| DESeq2          | 1.46.0  | @loveModeratedEstimationFold2014a                 |
| biomaRt         | 2.62.1  | @durinckBioMartBioconductorPowerful2005           |
| clusterProfiler | 4.14.6  | @yuClusterProfilerPackageComparing2012            |
| enrichplot      | 1.26.6  | @yuEnrichplotVisualizationFunctional2025          |
| msigdbr         | 24.1.0  | @dolgalevMsigdbrMSigDBGene2025                    |
| memes           | 1.14.0  | @nystromMemesMotifAnalysis2021                    |
| universalmotif  | 1.24.2  | @tremblayUniversalmotifPackageBiological2024      |
| decoupleR       | 2.12.0  | @badia-i-mompelDecoupleREnsembleComputational2022 |
| OmnipathR       | 3.14.0  | @valdeolivasOmnipathRClientOmniPath2019           |
| stringr         | 1.5.1   | @wickhamStringrSimpleConsistent2023               |
| Biostrings      | 2.74.1  | @pagesBiostringsEfficientManipulation2024         |
| dplyr           | 1.1.4   | @wickhamDplyrGrammarData2023                      |
| tidyr           | 1.3.1   | @wickhamTidyrTidyMessy2024                        |
| tibble          | 3.2.1   | @mullerTibbleSimpleData2023                       |
| ggplot2         | 3.5.1   | @wickhamGgplot2ElegantGraphics2016                |
| ggsci           | 3.2.0   | @xiaoGgsciScientificJournal2024                   |
| ggrepel         | 0.9.6   | @slowikowskiGgrepelAutomaticallyPosition2024      |
| pheatmap        | 1.0.12  | @koldePheatmapPrettyHeatmaps2019                  |
| ComplexHeatmap  | 2.22.0  | @guComplexHeatmapVisualization2022                |
| UpSetR          | 1.4.0   | @gehlenborgUpSetRMoreScalable2019                 |

: R packages used for the analysis and visualizations {#tbl-packages}

# Results

## Detecting Translation on Demand Candidates {#sec-detectingtod}

One hallmark of ToD regulation is a rapid increase in translation, while the rate of transcription stays stable. As a filtering criteria of ToD regulation, we selected only genes who showed RNA LFCs which stayed inside a stable range for each time interval (see @sec-todpara). The stringency of this range depends on the biological question we want to answer with a specific list of ToD candidates. For example, for gene set enrichment it might make sense to broaden the range we define as stable to detect functional trends in a certain time interval which would not be detectable otherwise. If not stated otherwise we used a stable LFC RNA range from minus one to one. The next filtering step to determine if a gene might be subject to ToD regulation at a specific time interval was to filter for a LFC-Ratio of at least one to ensure a substantial increase in protein levels while maintaining a relatively stable RNA LFC, which filters for robust protein level changes independent of RNA fluctuations (see @sec-todpara, @fig-tod_filtering A). Again, this LFC-Ratio threshold depends on the biological context and needs to be adapted for the dataset in question. If not stated otherwise, we used a LFC-Ratio threshold of one. This filtering step results in an initial ToD candidate list generated with the Yang Data of 113 ToD candidates for the first time interval (0h to 1h), 103 for the second (1h to 6h) and 78 for the third (6h to 12h) etc. (see @fig-tod_filtering B). Corrected for time, the number of ToD candidates are 103, 17 and 13 for the first three time intervals respectivly (see @fig-tod_filtering B). This trend continues in later time intervals. \newline To make the ToD candidate calling less subject to noise, we implemented a third filtering step. After a gene passes the LFC-Ratio threshold at a given time interval, we look at its protein LFC trajectory normalized for the first time point (0h) and see if the subsequent LFC value after the threshold passing is within a certain range defined by the time interval where it passed the LFC-Ratio threshold (@sec-todpara). For example, if a gene passed the LFC-Ratio threshold at the 0h to 1h time interval, we look in the protein LFC trajectory of that gene for 1h vs 0h and the next LFC value at 6h vs 0h. The LFC value at the 6h vs 0h time interval is only allowed to go down by 30% compared to the previous value at 1h vs 0h to eliminate genes that showed higher levels of noise in their protein LFC trajectory. After applying this filtering method, the number of ToD candidates for the first three time intervals were 61, 52, and 34 respectively (see @fig-tod_filtering C). The trend of having more ToD candidates when correcting for time in the early time intervals remained evident after applying the filtering step.

![**Filtering for ToD candidates in the Yang Data** \newline (A) Distribution of LFC-Ratios at different time intervals. The dashed red line (LFC-Ratio of 1) shows the LFC-Ratio threshold a gene needs to pass. The dashed blue line (LFC-Ratio of -1) shows the opposite behaviore where we see a decreased protein expression. \newline (B) The barplot shows the number of ToD candidates at each time interval which passed the LFC-Ratio of one. The dark green bars show the absolute number of ToD candidates at a given time interval. The light green bars show the number of ToD candidates corrected for time. \newline (C) The barplot shows the number of ToD candidates at each time interval which passed the LFC-Ratio and which passed the third filtering step. The dark green bars show the absolute number of ToD candidates at a given time interval. The light green bars show the number of ToD candidates corrected for time.](figures/yang_tod_plot.svg){#fig-tod_filtering fig-pos="H"}

## Early development is shaped by Translation on Demand candidates

Upon examining the filtered ToD candidates across various temporal intervals, it is evident that these candidates include numerous genes crucial for early developmental processes. Specifically, during the initial interval from 0 to 1 hour, a significant proportion of the ToD candidates are associated with developmental functions. Notably, genes such as Agap1, Dnajc14, Emc7, Rab27a, Vti1a and Napb play roles in protein trafficking and vesicular transport. \newline Not suprisingly then, genes associated with cytoskeletal regulation like Cdc42bpa, Cep95, Plekhh1 and Stil are represented among ToD candidates in the first time interval. \newline Another group of genes, such as Celf2, Rex2, and Rsrp1, are involved in post-transcriptional regulation. Specifically, Celf2 encodes an RNA-binding protein that plays a critical role in regulating several post-transcriptional events. It is involved in pre-mRNA alternative splicing, mRNA translation, and stability. Celf2 mediates exon inclusion and/or exclusion in pre-mRNA, which is subject to tissue specific and developmentally regulated alternative splicing [@laddCELFFamilyRNA2001]. Lastly, another group of genes like Axin1, Csnk1g2, Jag1, Rap2c and Rgs19 contributes to regulatory networks that govern developmental processes. \newline ToD candidates contributing to the mentioned processes can also be found in subsequent time intervals, like the splicing factor Srsf11 or developmental regulators like Epha2 in time interval 1h to 6h among others (see @supptbl-todyang).

## Gene over-representation analysis reveals time dependent pathway regulation of developmental terms by Translation on Demand candidates

Gene over-representation analysis (see @sec-clusterProfiler) revealed that ToD candidates are associated with with key developmental terms (see see @fig-ORA B). In the 0h to 1h time interval, we identified a nuanced overrepresentation of Notch signaling as well as Wnt/β-catenin signaling (see @fig-ORA A). \newline Additionally, in the 1h to 6h time interval ToD candidates associated with TNFA signaling and TGF beta signaling where significantly over-represented (see @fig-ORA B). \newline Next to these signalling Hallmark terms, we saw a significant over-representation of GO biological processes associated with development in that time interval (see @fig-ORA C). These included tissue specific developmental terms such as tube morphogenesis, vasculature development, epithelial cell proliferation, skeletal system development, ossification and response to growth factor. \newline ToD candidates of the 6h to 12h time interval also displayed a significant over-representation in GO molecular functions, specifically in calcium ion binding and calcium dependent protein binding as well as a nuanced over-representation in calcium dependent phospholipid binding (see @fig-ORA D). \newline Other significant gene over-representations were detected in the GO Cellular Components (CC) and Reactome during the 1h to 6h and 6h to 12h time intervals. CC terms included cellsubstrate junctions, golgi lumen as well as collagen containing extracellular matrix (see @suppfig-ORASUP). ToD candidate genes over-represented in reactome terms, where associated with extracellular matrix organization, glycoaminoglycan sythesis and apoptotic terms (see @suppfig-ORASUP).

![**Gene overrepresentation analysis of early ToD candidates in Yang Data** \newline (A) Hallmark term over-representation analysis at the 0h to 1h time interval. Adjusted p-value threshold of 0.1. Stable transcript range for ToD call of -2 to 2. \newline (B) Hallmark term over-representation analysis at the 1h to 6h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline (C) GO term over-representation analysis for biological processes (BP) at the 1h to 6h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline (D) GO term over-representation analysis for molecular function (MF) at the 6h to 12h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline The color gradient represents the adjusted p-value, with red indicating a lower adjusted p-value and blue indicating a higher adjusted p-value. The size of each dot corresponds to the number of genes present in the respective gene set from the dataset.](figures/ORA_plot.svg){#fig-ORA fig-pos="H"}

## Translation on Demand candidates regulate gene expression as transcription factors and their activity increases upon ToD regulation

As we identified a significant part (Fisher's Exact Test with pvalue of 0.021) of ToD candidate genes as transcription factors (TFs) using AnimalTFDB [@shenAnimalTFDB40Comprehensive2023], we wanted to investigate if these TFs display their activity upon potential ToD regulation or perhaps with a temporal offset after upregulation through ToD. Out of 438 filtered ToD candidates, 46 (10.5%) were identified as TFs within a stable transcript range of -2 to 2 (see @supptbl-todyang). From those 46 TFs, 22 had target annotation in CollecTRI (see @fig-tfheatmap). \newline Strikingly, most of the TF's activity derived from the transcriptomic data correlates with ToD regulation, meaning that they are most active upon ToD regulation and onward (see @fig-tfheatmap). Furthermore, these TFs are involved in the transition from the naive pluripotent to the primed formative state. \newline Foxd3 for example, which expresses a moderate to high activity in early time intervals and a potential ToD regulation at the 1h to 6h time interval (see @fig-tfheatmap). Klf10, which is potentially ToD regulated at the second time interval as well, exhibits a similar activity pattern (see @fig-tfheatmap). Additionally, Pbx1, ToD regulated in the second time interval, shows its highest activity during the early time points from 0h to 6h (see @fig-tfheatmap). Similarly, the expression pattern of Hmga1 aligns with that of Pbx1 (see @fig-tfheatmap). Other key factors for guiding the naive to formative transition like Sall2, Tbx3 and Tead1 are also present in ToD regulated TFs [@xiongSALL2RegulatesNeural2024; @khanRolesRegulationTBX32020; @nishiokaTead4RequiredSpecification2008]. \newline Notably Hsf2, Nr2c1 and Nfe2l2 showed a possible ToD regulation during the 1h to 6h time interval. During that time interval their activity as TFs is still moderate. After ToD regulation however they show elevated activity during intermediate time points from 24h to 36h. \newline

![**ToD TFs activity across samples in Yang Data** Activity scores of ToD candidate TFs present in CollecTRI network. Stable transcript range for ToD call of -2 to 2. Positive activity scores are colored in red. Negative activity scores are colored blue (see @sec-decoupler). Annotation heatmap (right) shows at which time interval(s) a specific TF is potentially ToD regulated.](figures/TF_heatmap.svg){#fig-tfheatmap fig-pos="H"}

## Translation on Demand candidates share RNA binding proteins target motifs

Now that we have established possible biological reasons why certain transcripts are subject to rapid selective translational upregulation, we wanted to understand mechanism underlying ToD. RBPs were a first possible contender since it is known that they influence mRNA translation by binding to transcripts inside or outside the nucleus. \newline To research if RBPs bind to our ToD candidates, we used AME from the MEME suite to do an enrichment analysis of RBP target motifs in 3' and 5' UTRs of ToD candidate transcripts across time intervals (see @sec-meme). We used the CISBP-RNA database to retrieve target motifs for RBPs [@rayCompendiumRNAbindingMotifs2013]. For the 3' UTR, 16 RBP target motifs were enriched in at least one time interval (see @fig-motifs A), while for the 5' UTR, 32 RBP target motifs were enriched in at least one time interval (see @fig-motifs B). Notably, most of the RBP target motif in the 3' UTR showed an enrichment in the early two time intervals (see @fig-motifs A), while target motifs in the 5' UTR showed the vast majority of enrichment in the 24h to 26h time interval (see @fig-motifs B). To verify the presence of these RBPs in the Yang Data and their potential impact on the ToD candidates, we examined the LFC time course of these specific proteins. In the proteomic data, 14 out of 16 RBPs for the 3' UTR and 23 out of 32 for the 5' UTR were identified (see @fig-motifs C, D). Notably, RBPs who have their target motif enriched in 3' UTRs of ToD candidate genes, displayed a more stable protein expression throughout the time course compared to RBPs who have their target motif enriched in 5' UTRs of ToD candidate genes (see @fig-motifs C, D). To check which ToD candidate genes' UTRs carry certain RBP target motifs and to see how the ratio of an enriched RBP target motif changes across time intervals, we searched for matching target motifs in the UTRs of ToD candidate genes. The results matched the enrichment behavior of the RBPs target motifs (see @suppfig-frac_motifs).

![**RBPs motifs in ToD candidate UTRs in Yang Data** \newline (A) AME results for RBPs target motifs enriched in 3'UTRs of ToD candidates across time intervals. All 3'UTRs sequences from matched Yang Data (present in proteome and transcriptome time course) were used as a control. Stable transcript range for ToD call of -2 to 2. \newline (B) AME results for RBPs target motifs enriched in 5'UTRs of ToD candidates across time intervals. Stable transcript range for ToD call of -2 to 2. All 5'UTRs sequences from matched Yang Data (present in proteome and transcriptome time course) were used as a control. \newline (C) LFCs of 3'UTR RBPs against 0h \newline (D) LFCs of 5'UTR RBPs against 0h](figures/motif_plot.svg){#fig-motifs fig-pos="H"}

\newpage

## Translation on Demand candidate calling in Leeb Data

To evaluate our method of calling ToD candidates, we wanted to apply our approach to other datasets to see if results are comparable and which challenges arise when dealing with different datasets. In our analysis of the Leeb TC Data, we observed a systematic dip in the LFC values of protein expression at the 4h time point. To account for this anomaly and ensure a more accurate representation of the underlying biological processes, we opted for a threshold of 0.5 instead of the our regular threshold of 1. With our three step filtering approach we detected five ToD candidate genes in the first time interval from 0h to 4h, 67 in the 4h to 8h time interval and 15 in the 8h to 12h time interval (see @suppfig-ratio_TC_MC). For the Leeb MC data we detected 288 ToD candidate in the time interval from 0h to 4h (see @suppfig-ratio_TC_MC). \newline Because we did not have matching time intervals in the three datasets, we compared early ToD candidate genes from 0h to 12h in the three datasets and found that they overlap in key developmental regulators. We detected seven early ToD candidates genes found both in the Yang Data and Leeb TC Data included Zfp36l1, Spry4, Epha2, Odc1, Bbx, Sfn, Parvb, Sfn and Flnc from which Odc1 was also found in ealry ToD candidate genes in the Leeb MC Data (see @fig-venn). Early ToD candidate genes found both in the Yang Data and Leeb MC Data included: Alg3, Bex1, Btbd9, Ebpl, Fam114a1, Farp1, Fzd7, G2e3, Nptx2, Odc1, Rps6ka6, Scd1, Sdc1, Slc30a1, Snx7 and Wdr90. This means that ornithine decarboxylase 1 (Odc1) was found in all three datasets in early ToD candidate genes.

![**Number of early ToD candidate genes and their overlap** The green, red and blue circle correspond to the Yang, Leeb TC and Leeb MC Data respectively. \newline Yang Data parameter: Stable transcript range: -2 to 2, ToD threshold: 1. \newline Leeb TC Data parameter: Stable transcript range: -2 to 2, ToD threshold: 0.5. \newline Leeb MC Data parameter: Stable transcript range: -2 to 2, ToD threshold: 1.](figures/early_tods_venn_diagram.svg){#fig-venn fig-pos="H"}

# Discussion

## Which genes are subject to ToD in early development?

With this study we aimed to identify which genes might be subject to ToD in early development. With our proposed filtering approach, we were able to identify genes, whos protein abundance rapidly increased during early development while transcript abundance remained stable. The fact, that we detect more ToD candidates per unit of time in early time intervals might be due to the relevance of ToD regulation during these early developmental time intervals, because rapid protein systhesis of specific transcipts is needed at that time to initiate priming for differentiation.

## Why are these genes regulated by ToD?

Our next objective was to illustrate why these genes are regulated by ToD. Looking at their gene function, it became clear that ToD candidates are deeply involved in key developmental processes. Among early ToD candidate genes we saw genes associated with protein trafficking, cytoskeletal regulation as well as posttranscriptional regulation like alternative splicing. This observation suggests that protein trafficking is likely subjected to substantial demand during this initial period, coinciding with the onset of protein translation following the removal of the 2i condition. The increased load on protein trafficking mechanisms may reflect the cellular necessity to rapidly establish the proteomic landscape required for subsequent developmental stages. These findings illustrate that the ToD candidates identified in the Yang dataset play a crucial role in facilitating the transition from the naive pluripotent state to the primed formative state. By contributing to essential processes such as protein trafficking, cytoskeletal regulation, post-transcriptional modification, and developmental signaling, these genes help orchestrate the complex molecular changes necessary for early development. \newline To gain more insight in which biological processes ToD candidate genes are involved in, we did gene ORA. In the first time interval from 0h to 1h, we detected a significant over-representation of ToD candidate genes in Notch signaling as well as Wnt/β-catenin signaling (see @fig-ORA A). Both pathways are well known for their involvement in embryonic development [@zhouNotchSignalingPathway2022; @liuWntVcateninSignalling2022]. The prominence of these pathways during the initial time interval suggests that they are among the first to respond to the removal of the 2i conditions. This response likely initiates a cascade of molecular events that prepare the cells for subsequent differentiation, highlighting their importance in the early phases of transitioning from a naive to a primed state. \newline During the next time interval from 1h to 6h, ToD candidates associated with TNFA signaling and TGF beta signaling where significantly over-represented (see @fig-ORA B). During embryogenesis and organogenesis, TNFA signaling plays a crucial role in mediating inflammatory responses and regulating apoptosis, which can influence tissue remodeling and development [@youTumorNecrosisFactor2021]. TNFA's involvement in these processes underscores its importance in maintaining cellular homeostasis and promoting the appropriate differentiation pathways necessary for proper tissue formation. Similarly, TGF-beta signaling is pivotal in embryonic development, as it regulates cell proliferation, differentiation, and migration. TGF-beta signaling is essential for the establishment of the body plan and the development of various tissues and organs [@dengTGFvSignalingHealth2024]. The overrepresentation of TGF-beta signaling in this interval suggests that ToD candidates may be actively contributing to the cellular transitions required for mESCs to exit the naive state and progress towards lineage specification. Furthermore, an over-representation during this time interval of tissue specific developmental terms suggest the progression toward priming for lineage commitment. This priming requires sophisticated and rapid reprogramming of the epigenetic, gene expression, and protein landscapes of ESCs. ToD might be a central mechanism enabling cells to make these necessary rapid changes. Again, CC terms such as Golgi lumen and substrate junctions in the second and third time interval (1h to 6h and 6h to 12h) may indicate an increase in protein trafficking and the initial progression towards the EpiSCs phenotype, respectively. Meanwhile, Reactome terms related to glycosaminoglycan (GAG) synthesis support this perspective by showing modulation of the extracellular environment and cellular interactions. Additionally, GAGs have been shown to have essential complementary roles in embryonic development [@kramerSpecificSidesMultifaceted2010]. The fact that we see an over-representation in calcium ion binding and calcium dependent protein binding, further paint the picture that ToD candidate genes aid mESC through the progression towards the primed formative state. Calcium-dependent signaling is a central and conserved mechanism essential for development by orchestrating key processes such as cell proliferation, differentiation, and migration. It plays a crucial role in various stages of embryogenesis, including the establishment of body plans, organogenesis, and tissue morphogenesis. The ability of calcium to act as a versatile second messenger allows it to integrate and decode diverse extracellular signals, thereby regulating the dynamic cellular environments necessary for proper development [@stewartElementDevelopmentCalcium2019].\newline These findings underscore the critical role that ToD candidate genes play in early development. The proteins encoded by these genes are essential for facilitating the transition from a naive to a formative primed state. This aligns with our initial hypothesis that ToD may serve as a cellular mechanism that enables swift alterations in the proteomic state of stem cells during development. By integrating cellular signaling cues, ToD effectively regulates gene networks to ensure a proper developmental transition. \newline This is further highlighted by the finding that TFs especially are subject to ToD. TFs like Foxd3, Klf10 and Pbx1, which are early ToD candidate genes play a critical role in progressing cell development. Foxd3 has been shown to promote exit from naive pluripotency by decommissioning enhancers and inhibiting germline specifications [@respuelaFoxd3PromotesExit2016]. Klf10 regulates target genes that impact tissue formation and may function in conjunction with other KLFs to sustain the stem cell state or propel differentiation [@bialkowskaKruppellikeFactorsMammalian2017]. The temporal activity behavior of Pbx1 agrees with the established relationship to NANOG, a naive marker. Pbx1 upregulates Nanog expression to sustain self-renewal and maintaining pluripotency [@chanKLF4PBX1Directly2009]. Later ToD candidate genes like Hmga1 are also involved in various aspects of development and differentiation, including cell proliferation and the regulation and maintenance of pluripotency [@vignaliHMGAGenesProteins2020]. This intricate regulation is crucial as mESCs transition from naive pluripotency towards the formative state. During this process, they must integrate developmental cues, which involves selectively dismantling the naive transcription factor circuitry while simultaneously acquiring competence for lineage commitment. It is therefore not surprising that ToD regulated TFs are also involved in this process, modulating the pluripotency network as the cells acquire lineage competence. TFs such as Hsf2, Nr2c1 and Nfe2l2 also play crucial roles next to developemntal regulation. During early development, ESCs are under increased cellular stress, and the selective upregulation of Hsf2 protein production could enhance the cell's ability to cope with these stressors by increasing the expression of heat shock proteins [@akerfeltHeatShockFactors2010]. Similar to Hsf2, Nfe2l2 is part of the cellular stress response, specifically in sensing environmental and endogenous oxidative stress. Deficiency of Nrf2 (the TF protein encoded by Nfe2l2) can lead to early embryonic lethality [@leungDeficiencyNrf1Nrf22003]. As part of the highly conserved nuclear hormone receptors (NHRs) superfamily, Nr2c1 is involved in many fundamental biological processes. Next to developmental and cell fate regulation, NHRs influence reproduction, metabolism, the circadian cycle, and immunological functions [@NuclearHormoneReceptor2017]. \newline Detected early ToD candidate genes in the Leeb TC and MC Data which were also present in the Yang Data datasets like Spry4, Rps6ka6 and Odc1 further support the picture that ToD regulation shapes early development. Especially Odc1, which was found as an early ToD candidate in all three tested datasets executes fundamental roles during embryonic development [@hussainExploringPolyaminesFunctions2017]. Polyamines during early embryonic development exhibit increased transcription of Odc1 in two cell mouse embryos, with expression levels rising throughout the blastocyst stages [@domashenkoExpressionMycfamilyMycinteracting1997]. Interestingly, there is evidence, which suggests modified polyamines can influence translation through eukaryotic initiation factor 5A (eIF5A) [@chatterjeeRapidDepletionMutant2006]. In summary, these results again show that ToD candidates fill critical roles in early ESC development. Most importantly by directly promoting the exit from naive pluripotency or supporting pluripotency to aid a proper transition to the primed formative state. Additionally genes like Hsf2, Nr2c1, and Nfe2l2 regulate cellular stress responses and developmental processes. Hsf2 and Nfe2l2 are particularly crucial for managing stress through heat shock and oxidative stress responses, respectively. These findings underscore the importance of ToD fine tuning gene expression to support critical transition mechanisms during early development.

## Which mechanisms enable ToD regulation?

Lastly, we wanted to explore which mechanisms might enable mESCs to facilitate ToD. Therefore we looked for RBP target motifs in UTRs of ToD candidate genes. Notably, these RBPs carry out essential roles in post-transcriptional regulation like mRNA stability, alternative splicing, translational efficiency and mRNA localization. In the first time interval from 0h to 1h we detected an enrichment for target motifs of the RBPs Rbmx, Rbm8a, Pprc1, Pcbp1, Pcbp2, Pcbp3, Lin28b and Hnrnpk (see @fig-motifs A). For example, among many other developmental functions, Lin28 interacts within large ribonucleoprotein complexes containing multiple RBPs and helicases to modulate translation of key developmental mRNAs, such as Dnmt3a and Hmga2 (a member of the same family we discovered as a ToD regulated TF, see @fig-tfheatmap) [@parisiIdentificationRNAbindingProteins2021]. Pcbp1-3 are part of the PCBP family, which are known to play a critical role in modulating mRNA stability, alternative splicing as well as translation regulation [@bakhmetPcbp1ConstrainsOct42024]. Furthermore, Pcbp1 was identified to timely downregulate Oct4 upon differentiation signals during the ESC transition from the naive to formative state [@bakhmetPcbp1ConstrainsOct42024]. In the second time interval from 1h to 6h we detected an enrichment for target motifs of the RBPs Tia1, Snrpb2, Rod1, Ptbp1, Mbnl1, Matr3 and Elavl2 (see @fig-motifs A). Again, RBPs like Ptbp1 orchestrate early embryonic development by integrating developmental cues in translational regulation and shape post-transcriptional modification by regulating alternative splicing [@liuRoleRNASplicing2023]. \newline Prominent groups of RBP target motifs enriched in 5' UTRs for example were PABPs like Pabpc1 and Pabpn1 which protect poly(A) tails from premature degradation, simultaneously stimulating translation initiation by facilitating the formation of a closed loop mRNA structure through interactions with cap binding proteins [@zhaoRevisitingPolyAbindingProteins2021]. \newline These results show that ToD candidates share RNA-binding protein (RBP) target motifs, which are crucial for rapid selective translational upregulation. The RBPs identified, such as Lin28b, Pcbp1-3, and Ptbp1, play pivotal roles in post-transcriptional regulation, affecting mRNA stability, alternative splicing, and translational efficiency. These proteins are integral to the regulatory networks that guide early embryonic development, responding to developmental cues to ensure precise gene expression and there might be a driving force behind ToD.

## Outlook

To further validate and expand our findings on ToD candidate genes, it is crucial to integrate additional datasets. By incorporating data from diverse sources and experimental conditions, we can strengthen the generalizability of our conclusions. This will also allow us to identify common mechanisms enabling ToD and candidate genes across different biological contexts and species. This is particularly important since the Leeb Datasets provide limited interpreterbility for the purpose of ToD calling. Although we found overlapping ToD candidate genes in early time intervals, additional datasets will be needed to conclude overarching trends. Mechansisms described by the Yang Data analysis are promising and will guide further analysis. \newline While our current study focuses on early development, ToD might also play a significant role in other biological processes such as stress responses, disease progression, and aging. Investigating ToD in these contexts could reveal novel regulatory mechanisms and therapeutic targets. Understanding how ToD operates in various physiological and pathological conditions will provide a comprehensive view of its biological significance. \newline To gain deeper insights into the translational regulation of ToD candidate genes, we are planning to employ polysome profiling. Polysome profiling allows for the examination of ribosome associated mRNAs, providing a more detailed understanding of translational dynamics. By identifying which transcripts are actively being translated, we can better understand the temporal regulation of protein synthesis during developmental transition stages. \newline ToD regulation in hESCs could differ from that in mESCs. Incorporating human datasets into our analysis will help identify species-specific ToD candidates and regulatory mechanisms, offering insights into human development and potential implications for regenerative medicine. Comparing human and mouse datasets will enhance our understanding of conserved and divergent ToD pathways across species. Ideally this might be one part of the question, why developmental time differs between the species.

# Conclusion

In this study we provide an approach to detecting Translation on Demand candidate genes by integrating multi-omics datasets including proteomics and transcriptomics. The findings of our study underscore the critical role of ToD candidate genes in early development, particularly in transitioning from a naive to a formative primed state. Furthermore, our study highlights the intricate regulation of protein synthesis during early development, identifying specific genes whose protein abundance rapidly increases despite stable transcript levels, suggesting that ToD is a crucial mechanism facilitating swift proteomic changes necessary for developmental transitions. \newline Finally, by adapting our approach to more biological contexts, we aim to deepen our understanding of the dynamic interplay between transcription and translation during critical developmental stages. This refinement could reveal novel regulatory pathways and therapeutic targets, enhancing our comprehension of developmental biology and its applications.

\newpage

# References

::: {#refs}
:::

\newpage

# Appendices

## Supplementary figures

![**Gene overrepresentation analysis of early ToD candidates in Yang Data** \newline (A) GO term over-representation analysis for cellular processes (CC) at the 1h to 6h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline (B) Reactome term over-representation analysis at the 1h to 6h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline (C) GO term over-representation analysis for cellular processes (CC) at the 6h to 12h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline (D) Reactome term over-representation analysis at the 6h to 12h time interval. Adjusted p-value threshold of 0.05. Stable transcript range for ToD call of -2 to 2. \newline The color gradient represents the adjusted p-value, with red indicating a lower adjusted p-value and blue indicating a higher adjusted p-value. The size of each dot corresponds to the number of genes present in the respective gene set from the dataset.](figures/sub_figures/GO_Reactome_Terms_subfig.svg){#suppfig-ORASUP suppfig-pos="H"}

![**RBPs motifs fractions in ToD candidate UTRs in Yang Data** \newline (A) RBPs target motifs fraction enriched in 3'UTRs of ToD candidates across time intervals. All 3'UTRs sequences from matched Yang Data (present in proteome and transcriptome time course) were used as a control. Stable transcript range for ToD call of -2 to 2. Fractions were calculated by dividing the number of 3' UTRs from ToD candidates carrying a specific motif at a given time interval by the total number of 3' UTRs from ToD candidates at that same time interval. \newline (B) RBPs target motifs enriched in 5'UTRs of ToD candidates across time intervals. Stable transcript range for ToD call of -2 to 2. All 5'UTRs sequences from matched Yang Data (present in proteome and transcriptome time course) were used as a control. Fractions were calculated by dividing the number of 5' UTRs from ToD candidates carrying a specific motif at a given time interval by the total number of 5' UTRs from ToD candidates at that same time interval.](figures/sub_figures/frac_RBPs_plot.svg){#suppfig-frac_motifs suppfig-pos="H"}

![**ToD calling in Leeb datasets** \newline (A) Distribution of LFC-Ratios at different time intervals in the Leeb TC dataset. The dashed red line (LFC-Ratio of 1) shows the LFC-Ratio threshold a gene would need to pass (adjusted ToD threshold 0f 0.5). The dashed blue line (LFC-Ratio of -1) shows the opposite behaviore where we see a decreased protein expression. \newline (B) Distribution of LFC-Ratios at different time intervals in the Leeb MC dataset. The dashed red line (LFC-Ratio of 1) shows the LFC-Ratio threshold a gene needs to pass. The dashed blue line (LFC-Ratio of -1) shows the opposite behaviore where we see a decreased protein expression. \newline (C) The barplot shows the number of ToD candidates in the Leeb TC dataset at each time interval which passed the LFC-Ratio and which passed the third filtering step. \newline (D) The barplot shows the number of ToD candidates in the Leeb MC which passed the LFC-Ratio and which passed the third filtering step.](figures/sub_figures/LFC_Ratio_plot_TC_MC.svg){#suppfig-ratio_TC_MC suppfig-pos="H"}

## Supplementary tables

::: {#supptbl-todyang tbl-colwidths="[20,80]" supptbl-pos="H" supptbl-cap="ToD candidate genes across time intervals in Yang Data" layout-nrow=2}

| Time interval | ToD candidate genes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 0h to 1h      | Abhd14a, **Agap1**, **AI987944**, Ankrd35, **Arhgap27**, Asap3, **Axin1**, **Bbs5**, **Bbs9**, **Bex4**, Brat1, Bst2, **Btbd9**, **Cacna1a**, **Cbx6**, Ccdc136, **Ccdc91**, **Ccnf**, **Cdc42bpa**, Cebpg, **Celf2**, **Cep95**, Cetn4, **Cgn**, Cobll1, Coro2b, **Csnk1g2**, **Csrnp2**, Ctu1, Dapk3, **Dcbld1**, Dcp1b, Dgkh, Dhrs13, Dixdc1, **Dnajc14**, **Ebpl**, **Emc7**, Eml1, **Enox2**, **Epha1**, **Fbxo9**, Fn1, Fzd7, **Gba2**, Gbx2, Gm10479, **Gm5141**, Gng2, Gramd1a, Gramd1b, **Gstp2**, **Hps4**, Igsf3, Iqsec2, Irf2bp2, Jade2, **Jag1**, Kifc3, Klhl25, Lsm11, Map2, Mgst3, Mlycd, Mrm1, Msi1, Mtmr4, Myh7b, **Napb**, Napepld, Nfatc2, **Nudt22**, **Odc1**, Osbpl7, **Otud5**, Pabpc1, Parvb, Pdk2, Pfkfb3, Pla1a, **Plch1**, **Plekhh1**, Ppard, Ppard, Prkcz, Proser1, Ptk2b, **Rab27a**, **Rap2c**, Rasal1, Rasgrp2, **Rassf1**, Reep6, **Rex2**, **Rgs19**, **Rhbdd3**, **Rnf187**, **Rsrp1**, Sbk1, Shb, Sipa1l3, **Slc38a2**, **Smim20**, **Sowahb**, Srcin1, Srrd, Stard5, Stard9, **Stil**, Syngr1, **Taf11**, Tead1, **Tex15**, Tfap4, **Tm2d3**, **Tmc6**, Tmem161a, **Tmub2**, **Tnks**, **Tpst2**, Trh, Tsc22d4, **Vangl1**, Vax2, Vmac, **Vti1a**, Wdr90, Ybx2, Zbed3, **Zcchc2**, **Zfp850**, **Zik1**, **Zmym6** |
:::

::: {#tbl-abbr table.cap = NULL tbl-colwidths="[20,80]"}
| Time interval | ToD candidate genes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 1h to 6h      | **Agtrap**, **Alg3**, Aox3, **Arhgap30**, Armc5, **Avpi1**, Bex1, **Bmpr1a**, Btg3, Cap2, **Capn3**, **Ccdc112**, **Cdh1**, **Coro2b**, Cox7a1, **Ctdsp2**, **Dip2c**, Dsg2, Dym, **Epha2**, Fam114a1, **Fam168b**, Farp1, **Fat1**, Foxd3, **Fzr1**, **G2e3**, Gm21992, Gpc4, Hmox1, **Hsf2**, Hspg2, Id1, **Ifrd1**, **Irs2**, **Itgb3**, **Jak1**, **Kifc5b**, Klf10, Klhl7, **Lgals8**, **Lin28b**, Lpl, **Maml1**, **Mapkapk2**, **Micall1**, Nab1, Nenf, Neo1, **Nfe2l2**, Nodal, **Nptx2**, **Nr2c1**, Odc1, P2rx7, **Pbx1**, **Pde8a**, **Pecam1**, Ppp1r15b, **Rimklb**, **Rnf187**, **Rps6ka6**, Sdc4, **Slc19a2**, Slc30a1, **Slc9a8**, Smoc1, Spry4, **Sptb**, **Sqstm1**, **Srsf11**, Srsf5, **Ss18l2**, St14, **Sulf1**, **Suv39h1**, **Syde2**, Tbx3, **Tex15**, **Tmem106b**, **Traip**, Trak1, **Trim37**, **Trmt44**, Vcan, Zfand2a, **Zfand5**, Zfp36l1, **Zfp84**, Zic2, Zyx |
| 6h to 12h     | 2200002D01Rik, Adam23, **Adck2**, **Akap11**, Anxa1, Anxa2, Anxa3, Arhgdib, **Bbx**, **Bcl2l11**, **Bex4**, **Birc2**, **Bnc2**, Celf2, **Cenpj**, **Cers5**, **Cox7a2**, Dnmt3a, **Dzip1l**, **Eif1b**, **Enox2**, Fam114a1, Flnc, Frmd4b, **Gpc4**, Gsn, Gsto1, **Htra1**, Itgb1bp1, Itgb1bp1, **Kctd9**, **Klhdc10**, Lamb1, Lmna, **Lnx2**, Lrp2, Lrrfip1, Mageb4, **Mpped2**, **Myo1d**, Nav2, Nedd4l, Nt5e, **Ormdl3**, Osbpl3, **Parp9**, Pdzd4, Phldb1, Plec, **Pnp**, Ptk2b, Ptpn12, Ptpn13, **Rab40c**, **Rapgef2**, Rasgrp2, **Rbm44**, **Rims2**, **S100a10**, S100a6, Sall2, **Scd1**, **Sdc1**, Sfn, Sh3pxd2b, **Slc25a31**, Snx7, Tktl2, **Tmem17**, Ttc21b, **Ttll4**, **Ube2q2**                                                                                                                                                                                                |
| 12h to 24h    | 2200002D01Rik, Acsl6, **Alox15**, Anxa3, Arhgdib, **Arhgef25**, **Bbc3**, **Camkv**, Capn3, Ccdc136, Cgn, Cryab, **Dapk1**, **Dgkd**, Dnaja4, Dnmt3a, Enox1, Enpp3, Fam83h, Grhl2, Ift122, Ina, Krt8, Lrp2, **Mapk6**, **Mief1**, **Mthfsd**, Nav2, **Nuak2**, **Phf6**, **Pop5**, Psmb10, **Rab27a**, S100a6, **Sh3kbp1**, **Slc35b1**, **Spop**, Spta1, Stat5b, Tgm3, Thy1, **Tktl2**, **Trim23**, **Zfp644**, Zmym6                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 24h to 36h    | **Actn3**, Adck2, Ankrd35, Arhgdib, **Asnsd1**, Atp9a, Camk1, **Capn3**, **Chchd7**, **Cnn2**, Colec12, **Dgkd**, Dnmt3l, Dusp16, **Enpp3**, Epas1, Fam184a, **Fam83b**, **Fbxw17**, Fkbp10, **Foxp1**, Gm21992, **Gng3**, Gpsm1, **Gstm4**, **Hmga1**, **Hmox1**, Klhl24, **Lnx2**, Map1lc3a, Mcf2l, Mlxipl, **Mrpl52**, **Nr2c1**, **Otub2**, P4ha2, Pdzd2, **Peli1**, **Plce1**, **Pop5**, Ppp1r9a, Psmb10, **Rsl24d1**, Sirt4, **Siva1**, Slc9a7, **Spta1**, **Srgap1**, **Sumo3**, **Syne1**, Usp26, **Zfp644**                                                                                                                                                                                                                                                                                                                                                                             |
| 36h to 48h    | Abhd14b, Armcx3, **B4galt1**, **Clip2**, Cobll1, Crabp2, **Cthrc1**, Enpp3, **Fam222b**, **Flnc**, **Flywch1**, Gbx2, Gdap1l1, Glcci1, **Glt1d1**, Grhl2, Hdac7, **Irs2**, Lin28b, Mapk12, **Mlxip**, **Mutyh**, **Phykpl**, Prr36, Prtg, Ptk2b, **Ptpn13**, Rasgrp2, **Rbm47**, Rgs12, **Rragd**, Sema4d, Setd7, Sfmbt2, Slc16a3, Slc25a12, Smpdl3b, Sox3, **Tmem245**, **Trio**, Vcan, Vrtn, **Wwp1**, Zc4h2, Zfp125                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
:::


## Code and Data availability

### Datasets

The transcriptomic time course data from the Yang Data is available on the Gene Expression Omnibus database under: [GSE117896](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE117896). The corresponding proteome time course data is available on the ProteomeXchange Consortium under: [PXD010621](https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD010621) [@yangMultiomicProfilingReveals2019]. Transciptomic time course data from the Leeb Data is available on the Gene Expression Omnibus database under: [GSE145653](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145653) [@lacknerCooperativeGeneticNetworks2021]. The corresponding time course proteome data and the Leeb Medium Change dataset can be requested.

### MEME analysis docker container

For the motif analysis we used the MEME suite alongside R. Therefore we set up a docker container which had the MEME CLI tool installed as well as R and the required R packages to run the motif analysis. The docker image file is accessible at the [Beyer Group GitHub repository](https://github.com/beyergroup/memeR_docker).

### R package for calling Translation on Demand candidates: ToDcallR

Because we had to do many iterations of how to filter for ToD candidates with different parameters and datasets, we developed ToDcallR alongside to help us with our analysis, making our methods more accessible and to make it easier for future projects to call potential ToD regulated genes. The package is still under development, because the constant refinement of our calling methods. If you want to use the package or want to contribute, you can access the package on my [GitHub repository](https://github.com/eliasschwall/ToDcallR).

### Translation on Demand exploration dashboard

If you want to look and LFC-Ratios of single genes and see their LFC time course across time intervals you can go to Translation on Demand exploration dashoard.

## Abbreviations


::: {#tbl-abbr table.cap = NULL}
| Abbreviation | Definition                                  |
|--------------|---------------------------------------------|
| ToD          | Translation on Demand                       |
| mESCs        | Mouse embryonic stem cells                  |
| PSCs         | Pluripotent stem cells                      |
| EpiSCs       | Epiblast stem cells                         |
| hESCs        | Human embryonic stem cells                  |
| LIF          | Leukemia inhibitory factor                  |
| ESCs         | Embryonic stem cells                        |
| HSPs         | Heat shock proteins                         |
| eIF4E        | Eukaryotic translation initiation factor 4E |
| MNKs         | MAPK-Interacting Kinases                    |
| RNPs         | Ribonucleoproteins                          |
| UTR          | Untranslated region                         |
| uORFs        | upstream Open Reading Frames                |
| IRESs        | Internal Ribosome Entry Sites               |
| miRNAs       | mircoRNAs                                   |
| RBPs         | RNA-binding proteins                        |
| AREs         | AU-rich elements                            |
| RBDs         | RNA-binding domains                         |
| IDRs         | Intrinsically Disordered Regions            |
| RBPs         | Poly(A)-Binding proteins                    |
| LFC          | Log Fold Change                             |
| GPR          | Gaussian Process Regression                 |
| TC           | Time Course                                 |
| MC           | Medium Change                               |
| LFC-Ratio    | Protein-to-RNA-LFC-Ratio                    |
| GO           | Gene ontology                               |
| BP           | Biological processes                        |
| MF           | Molecular function                          |
| CC           | Cellular components                         |
| GAG          | Glycosaminoglycan                           |
| TFs          | Transcription factors                       |
| ULM          | Univariate Linear Model                     |
| NHRs         | Nuclear Hormone Receptors                   |
| Odc1         | Ornithine Decarboxylase 1                   |
| eIF5A        | Eukaryotic initiation factor 5A             |
:::

