---
title: "Translation on Demand as a post-transcriptional regulation mechanism of ES cell differentiation"
subtitle: "Master thesis for acquisition of the degree of Master of Science (M.Sc.) in Computational Biology of the faculty of mathematics and natural sciences at the University of Cologne"
author: "submitted by Elias Schwall"
date: "today"
date-format: long
fontsize: 12pt
mainfont: "Helvetica"
geometry: margin=2.5cm
header-includes:
- \usepackage{authblk}
- \usepackage[fontsize=12pt]{scrextend}
- \usepackage{setspace}
- \setstretch{1.5}
- \usepackage{graphicx}
- \pagenumbering{gobble}
- \usepackage{fancyhdr}
- \pagestyle{fancy}
- \fancyhf{}
- \fancyfoot[R]{\thepage}
format: 
  pdf:
    documentclass: article
    classoption: "12pt"
    link-citations: true
bibliography: Master_Thesis.bib
---

```{=latex}
\textbf{Reviewers}: \newline
Prof. Dr. Andreas Beyer \newline
Prof. Dr. Achim Tresch

\textbf{Supervised by}: \newline
Luke Brandwood

\vspace*{\fill}
\vspace*{-2cm}  
\begin{figure}[h]
    \centering
    \begin{minipage}{0.5\textwidth}
        \centering
        \includegraphics[width=\textwidth]{CECAD_logo.jpg}
    \end{minipage}\hfill
    \begin{minipage}{0.4\textwidth}
        \centering
        \includegraphics[width=\textwidth]{Uni_Köln_Logo.png}
    \end{minipage}
\end{figure}

\vspace*{\fill}

\newpage

\pagenumbering{Roman}

\section*{Declaration of originality}

I hereby declare that this thesis is my own work and that no other sources and tools than stated were used.\newline

\vspace*{\fill}

Cologne, \today \hspace*{\fill} Elias Schwall

\newpage

\section*{Acknowledgement}

\newpage

\tableofcontents

\newpage

\setcounter{page}{1}
\pagenumbering{arabic}
```

# Abstract

\newpage

# Introduction

## Mouse embryonic stem cells

Mouse embryonic stem cells (mESCs) are defined by their self-renewal capabilities and pluripotency, allowing them to differentiate into any cell type from the three germ layers. While this differentiation process is limited in vivo, pluripotent stem cells (PSCs) cultivated in vitro can self-renew indefinitely under specific culture conditions [@nicholsNaivePrimedPluripotent2009]. One such condition is the 2i+LIF (two inhibitors with leukemia inhibitory factor) medium, which maintains the naive pluripotent state of ESCs, closely mimicking the inner cell mass of pre-implantation mouse embryos. \newline The continuum of pluripotency delineates a progression from the naive state to a primed state, characterized by an increased propensity for differentiation [@nicholsNaivePrimedPluripotent2009]. Naive mESCs serve as a model for the inner cell mass of the pre-implantation embryo, exhibiting minimal lineage commitment. In contrast, epiblast stem cells (EpiSCs), which are isolated from later developmental stages, correspond to the post-implantation epiblast and are primed for differentiation [@boroviakAbilityInnercellmassCells2014a]. \newline During the transition from the naive to the primed states, sequential changes occur at multiple molecular levels, encompassing alterations in signaling pathways, epigenetic markers, and gene expression [@dejosezPluripotencyNuclearReprogramming2012a]. Specifically, early modifications in the phosphoproteome serve as indicators of these state transitions, initiating a cascade of changes that ultimately impact the transcriptome and proteome [@needhamIlluminatingDarkPhosphoproteome2019a]. This progression underscores the dynamic nature of pluripotency and highlights the intricate molecular mechanisms governing cell fate decisions in early development. \newline It is worth noting that early mESCs development differs from human embryonic stem cells (hESCs). While the molecular mechanisms controlling developmental processes are evolutionarily conserved, the speed at which these processes occur can vary significantly between species. For instance, the differentiation of motor neurons from embryonic stem cells is more than twice as fast in mice compared to humans [@rayonSpeciesspecificDevelopmentalTiming2020]. \newline Regardless of the species, however, early developing embryonic stem cells need to undergo rapid changes in their gene expression profiles to ensure proper development and differentiation. Mechanisms such as ToD, can allow for swift adjustments in protein levels to aid these rapid changes. This ensures that cells can quickly respond to developmental cues and environmental changes, maintaining the necessary pace of development. Such mechanisms are crucial for adapting to the specific temporal requirements, ensuring that embryonic stem cells can effectivly progress through the stages of development.

## Translation on demand

We define ToD as an increase in protein abundance due to enhanced translation, occurring without any change in the mRNA levels of the corresponding gene. In other words, ToD is a regulatory mechanism where translation efficiency is selectively increased for specific genes. This upregulation allows for rapid adjustments in protein levels, supporting cellular responses that need immediate protein synthesis without requiring new mRNA transcription. Such translation driven increases could be particularly useful in situations where quick adaptations are necessary, as it enables cells to respond to changes in the environment or developmental cues by rapidly elevating the protein output from existing mRNA pools. \newline In vivo, there are already described mechanisms that facilitate forms of selective, rapid translation of mRNAs. For example, neurons need local translation at active synapses for quick cellular responses [@holtCentralDogmaDecentralized2013]. In early embryos, maternal mRNAs are stored and activated after fertilization, enabling the embryo to rapidly synthesize proteins necessary for early development without new mRNA synthesis [@deleonHalflivesRelativeAmounts1983]. In stress response, rapid and selective synthesis of stress proteins, such as heat shock proteins, is facilitated by ToD, allowing cells to quickly adapt and protect themselves under stress conditions without increasing mRNA levels [@holcikTranslationalControlStress2005].

## 2i Condition for maintaining naive pluripotency

Since serum contains undefined factors that can induce differentiation, the 2i medium was developed to provide a defined, chemically controlled environment that preserves mESCs in the naive pluripotent state [@yingGroundStateEmbryonic2008]. The 2i medium relies on the synergy between two inhibitors, PD0325901 and CHIR99021, to sustain mESCs in an undifferentiated state. PD0325901, a MEK inhibitor, blocks the MEK/ERK signaling pathway, suppressing differentiation signals from FGF and ERK that would otherwise prompt mESCs to exit pluripotency. Meanwhile, CHIR99021, a GSK3α/β inhibitor, stabilizes β-catenin and activates Wnt/β-catenin signaling, promoting the expression of self-renewal genes and reinforcing pluripotency [@yingGroundStateEmbryonic2008]. Together, these inhibitors create a controlled environment that preserves mESCs in a ground state resembling the undifferentiated pre-implantation epiblast [@yingGroundStateEmbryonic2008]. Alongside leukemia inhibitory factor (LIF), which activates the JAK-STAT3 pathway to further inhibit differentiation, the 2i medium maintains mESCs in a naive pluripotent state, enabling them to retain full differentiation potential without lineage bias [@yingGroundStateEmbryonic2008]. \newline The removal of the 2i medium triggers the transition from the naive pluripotent state by reactivating the MEK/ERK and GSK3 pathways, allowing ERK signaling to promote differentiation and reducing Wnt signaling to enable lineage specification. This shift initiates a cascade of biological changes, including the downregulation of naive markers (such as Nanog and Esrrb) and upregulation of formative markers (such as Otx2), along with epigenetic remodeling, altered chromatin accessibility, and a metabolic shift from glycolysis to oxidative phosphorylation [@wrayInhibitionGlycogenSynthase2011a].

![**2i mechanism of stabilizing naive pluripotency** The inhibition of Mek/Erk through 2i-dependent mechanisms prevents the phosphorylation of Erf and Cic, resulting in the stabilization of these factors within the nucleus. These transcriptional effectors may play a role in activating genes associated with naïve pluripotency while repressing those involved in primed/formative states. Cdk8/19i influences the phosphorylation of proteins that are part of the elongation and transcription machinery, including the super elongation complex (SEC), RNA polymerase II (RNPII), and Mediator (MED). This could lead to the rapid activation of genes regulated by super-enhancers, which include factors that stabilize the naïve state as well as certain factors from the primed/formative states that are also upregulated in response to Cdk8/19i. The activation of the naïve circuitry through 2i and Cdk8/19i may support the enhanced mitochondrial oxidative phosphorylation (OXPHOS) capacity of these cells. The degradation of Lin28a might alleviate its repressive effects on target mRNAs, leading to the upregulation of mitochondrial proteins among others. The balance between autophagy and protein synthesis may sustain the translation of these mitochondrial components. Additionally, the re-wiring of mitochondrial sources for one-carbon units may further support the translation of mitochondrial-encoded proteins. Furthermore, the increased availability of S-adenosylmethionine (SAM) levels, due to NNMT upregulation, combined with the impairment of DNA methylation caused by Uhrf1 degradation, contributes to the characteristic hypomethylated DNA state typical of 2i conditions. [@martinez-valDissectionTwoRoutes2021]](figures/2i_Figure.png){#fig-2i fig-pos='H'}

## ERK signalling pathway and its effects on translation

## mTOR signalling pathwy and its effect on translation

![**ERK and mTORs effect on translation** [@uttamEIF4EDependentTranslationalControl2018]](figures/ERK_and_mTOR_effect_on_translation.jpg){#fig-erg_and_mTOR fig-pos='H'}

## Additional mehcanisms infuencin translation

## Aim of this study

In this project, we aim to identify ToD candidates from a multi-omic dataset, which includes a time series of bulk RNA sequencing and proteomic data [@yangMultiomicProfilingReveals2019]. These multiple omic layers enable us to explore how protein abundance responds to changes in mRNA levels and to identify genes that may be regulated through ToD mechanisms. Next to finding genes that are subject to ToD regulation, we want to illustrate why these specific genes are prioritized and to uncover the underlying mechanisms of this regulation. This involves exploring the biological roles and functional significance of these genes in cellular processes. Additionally, we are interested in understanding the gene regulatory network and proteome changes that occur as mESCs exit pluripotency, as these regulatory networks are most likely to be regulated by ToD. \newline By analyzing the temporal dynamics within these data layers, we can distinguish cases where protein levels increase independently of mRNA changes, a hallmark of ToD regulation. This approach provides a comprehensive view of the post-transcriptional mechanisms that contribute to protein abundance, offering insights into gene-specific regulatory processes. Understanding the more nuanced relationship between RNA and protein expression, as well as the timing of regulation, will provide valuable insights into the fundamental principles of molecular biology and enhance our comprehension of cellular regulatory networks as well as early development. \newline Furthermore, this study should serve as a framework to detect ToD candidates in general. We also aim to validate our findings and approaches with a human temporal dataset to check the validity of the methods and see if we obtain similar results. This cross-species validation will help ensure that the mechanisms identified are robust and applicable across different biological contexts.

## 

# Methods and tools

## Data overview and processing

## Translation on demand metrics

## RNA-Seq

## Proteomics

## Ribo-Seq

## Gene Overrepresentation Analysis

## Gene set enrichment analysis

## Motif enrichment

## Software

# Results

## Early developement is shaped by ToD

## Gene overrepresentation analysis reveals time dependent pathway regulation of developmental terms

## ToD candidates regulate gene expression as transcription factors

## ToD is influenced by RNA binding proteins

## GSEA reveals

## Riboseq reveals

# Discussion

# Conclusion

\newpage

# References

::: {#refs}
:::

\newpage

# Appendices

## Supplementary figures

## Supplementary tables

## Code and Data availability

\newpage

## Abbreviations

| Abbreviation | Definition                 |
|--------------|----------------------------|
| ToD          | Translation on demand      |
| mESCs        | Mouse embryonic stem cells |
| PSCs         | Pluripotent stem cells     |
| EpiSCs       | Epiblast stem cells        |
| hESCs        | Human embryonic stem cells |
| LIF          | Leukemia inhibitory factor |
|              |                            |

: List of abbreviations
