---
title: "Translation on Demand as a post-transcriptional regulation mechanism of embryonic stem cell differentiation"
subtitle: "Master Thesis Defense"
author: "Elias Schwall"
loop: true
date: today
logo: CECAD_logo.jpg
css: style.css
format: 
  revealjs:
    slide-number: true
include-in-header:
  - text: |
      <style>
      #title-slide .title {
        font-size: 1.5em;
      }
      </style>
include-after: |
  <script type="text/javascript">
    Reveal.on('ready', event => {
      if (event.indexh === 0) {
        document.querySelector("div.has-logo > img.slide-logo").style.display = "none";
      }
    });
    Reveal.addEventListener('slidechanged', (event) => {
      if (event.indexh === 0) {
        Reveal.configure({ slideNumber: null });
        document.querySelector("div.has-logo > img.slide-logo").style.display = "none";
      }
      if (event.indexh === 1) { 
        Reveal.configure({ slideNumber: 'c/t' });
        document.querySelector("div.has-logo > img.slide-logo").style.display = null;
      }
    });
  </script>
---



## What is Translation on Demand (ToD)? {.smaller}

:::::: columns
:::: {.column width="60%" .incremental}
::: callout-note
# ToD

is described as a **selective** and **rapid** protein synthesis for specific mRNAs **without increased mRNA levels**
:::

-   quick cellular responses during transitions, stresses and signaling stumuli
-   **Neurons:** Local translation at active synapses
-   **Early Embryos:** Stored maternal mRNAs are activated after fertilization
-   **Stress Response:** Rapid synthesis of stress proteins (e.g. heat shock proteins) when needed
::::

::: {.column width="40%"}
![](figures/defense_specific/Translation_on_Demand_Info_graphic.png){height="600"}
:::
::::::

## Why should we care?

:::{.r-stack}

:::{.fragment .fade-in}
:::{.fragment .fade-out}
::: callout-tip
## Transcript-Protein Relationship
RNA levels can not fully explain protein abundance for many genes
:::

![](figures/defense_specific/Transcript_RNA_relation.png)
:::
:::

:::{.fragment .fade-in .incremental}
::: callout-tip
## Mouse vs Human Development
- mESCs differentiate/develop much faster than hESCs 
- maybe ToD is a key mechanism that allows this speed 
- e.g. enabling the upregulation of key genes to switch gene regulatory networks quickly
:::

:::
:::


## [ToD may be a key regularoty machanism during development]{.r-fit-text}

:::{.r-stack}
![edited from Strell et al. 2022](figures/defense_specific/ESCs_1.png){width=90%}

![edited from Strell et al. 2022](figures/defense_specific/ESCs.png){.fragment width=90%}
:::


## How are we studying ToD? - 2i Media {.smaller}

::::: columns
:::: {.column width="60%"}

::: {.fragment}
::: callout-caution
## <span style="color: green">GSK3 Inhibitor (CHIR99021)</span>
- Inhibits GSK3 to stabilize β-catenin, promoting Wnt signaling
- Supports pluripotency and helps prevent differentiation
:::
:::

![Fabian Titz-Teixeira PhD Thesis](figures/defense_specific/2i_media.png)
::::
 
:::: {.column width="40%"}

::: {.fragment}
::: callout-caution
## <span style="color: orange">MEK Inhibitor (PD0325901)</span>

- Blocks the ERK pathway to decrease differentiation signalling
:::
:::

::: {.fragment}
::: callout-tip
## The basic idea
Remove 2i → Perform –omics over a timecourse → Analyze changes in genes and their products
:::
:::

::::

:::::

## Project Goals

::: {layout-nrow=2}
::: callout-tip
# [1] Identifying ToD regulated genes
Framework for filtering ToD candidates by comparing transcriptome and proteome layers
:::

::: {.fragment}
::: callout-caution
# [2] What processes are subject to ToD?
Define gene sets / Functional enrichment and annotation 
:::
:::

::: {.fragment}
::: callout-caution
# [3] Why are these processes ToD-regulated?
Are those processes critical for transitioning out of the naive state?
:::
:::

::: {.fragment}
::: callout-caution
# [4] How is ToD facilitated by cells?
What are the mechanisms enabling ToD? E.g. regulatory elements in 5' UTRs, 3' UTRs, RNA binding proteins, etc.
:::
:::
::: 

## [Experimental Approach, Datasets and Workflow]{.r-fit-text}

::::: columns
:::: {.column width="80%"}
![](figures/defense_specific/Master_Thesis_Project_Overview.png)
:::: 
:::: {.column width="20%"}

:::: 
:::::

:::{.fragment}
![](figures/defense_specific/ToDcallR_hex.png){.absolute bottom=150 right=-70 width="380" height="300"}
:::

## [How can we detect ToD candidates?]{.r-fit-text}{.smaller}

::: callout-note
## Filter for stable transcripts
$$ -R_s \leq \text{LFC}_{\text{between time points}} \leq R_s$$
:::

::::: columns
:::: {.column width="50%"}

::: {.fragment fragment-index=1}
![](figures/defense_specific/stable_range.png){width=70%}
:::

::::
:::: {.column width="50%"}

::: {.fragment fragment-index=2}
```{r}
#| eval: false
#| echo: true
#| code-line-numbers: "|5"
DESeq2::results(
      ToDcall@dds,
      contrast = contrasts[[comparison]],
      lfcThreshold = stable_transcript_range,
      altHypothesis = "lessAbs"
)
```
:::
::::
::::: 




## [How can we detect ToD candidates?]{.r-fit-text}{.smaller}


::: callout-note
## Calculate the LFC ratio
$$\text{(1) LFC-Ratio} = \text{logFC}_{\text{between time points protein}} - \text{logFC}_{\text{between time points RNA}}$$
$$\text{(2) LFC-Ratio}\geq \text{ToD-threshold}$$
:::


:::{.fragment}
![](figures/defense_specific/yang_ratio_plot.svg)
:::

## [How can we detect ToD candidates?]{.r-fit-text}{.smaller}

::: callout-note
## Look at protein trajectory
$$\text{logFC}_{\text{protein against 0h after ToD call}}\geq \text{logFC}_{\text{protein against 0h during ToD call}} \times (1-R_d)$$
:::

:::{.fragment}
![](figures/defense_specific/LFC_good_bad.png)
:::

## [Early time intervals experience more ToD]{.r-fit-text}
![](figures/defense_specific/tod_numbers.svg)

## Early ToD candidates at a glance
::: {layout-nrow=2}

:::{.fragment}
:::{.callout-note .smaler}
## Vesicle budding, endo-/exocytosis and membrane trafficking

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">Agap1, Dnajc14, Rab27a, Vti1a, Napb</p>
:::
:::

:::{.fragment .smaller}
::: callout-note
## Cytoskeletal re-organisation, centrosome & centriole functions

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">Cdc42bpa, Cep95, Plekhh1, Stil</p>
:::
:::

:::{.fragment .smaller}
::: callout-important
## Post-transcriptional control (splicing, RNA stability)

<p style="text-align:center; color:#DD3545; font-weight:bold;">Celf2, Rex2 (REXO2), Rsrp1</p>
:::
:::

:::{.fragment .smaller}
::: callout-tip
## Signal-integration hubs (Wnt/ß-catenin, Notch, small-GTPase & G-protein regulation)

<p style="text-align:center; color:#1B8754; font-weight:bold;">Axin1, Csnk1g2, Jag1, Rap2c, Rgs19</p>
:::
:::

:::

## [ToD candidates are over-represented in key transition processes]{.r-fit-text}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/ORA_sig.png)
::::
::::{.column width="40%"}
:::{.fragment}
:::{.callout-important}
## Signaling

- **TNFA**: Inflammatory/apoptotic remodeling
- **TGF-ß**: Proliferation, migration, tissue specification

<p style="text-align:center; color:#DD3545; font-weight:bold;">Exit from naïve state → lineage priming </p>
:::
:::
::::
:::::

## [ToD candidates are over-represented in key transition processes]{.r-fit-text}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/ORA_dev.png)
::::
::::{.column width="40%"}
:::{.fragment}
::: callout-note
## Development

Tube morphogenesis, vasculature development, epithelial cell processes, skeletal development

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">Tissue programme priming</p>
:::
:::
::::
:::::

## [ToD candidates are over-represented in key transition processes]{.r-fit-text}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/ORA_cell.png)
::::
::::{.column width="40%"}
:::{.fragment}
::: callout-note
## Cellular Machinery

Cell surface, substrate junctions, Golgi lumen

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">Enhanced protein trafficking</p>
:::
:::
::::
:::::

## [ToD candidates are over-represented in key transition processes]{.r-fit-text}{.smaller}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/ORA_mat.png)
::::
::::{.column width="40%"}
:::{.fragment}
::: callout-tip
## Matrix & GAGs (Glycosaminoglycans)

- **Cell-surface proteoglycans**: Co-receptors & morphogen reservoirs
- **Heparan-sulfate**: Required for FGF2-ERK & BMP4 → pluripotency exit
- **Chondroitin-sulfate**: E-cadherin binding → cell-cell adhesion
- **Sulfation patterns**: Fine-tune Wnt, BMP, Hedgehog, TNF-a gradients

<p style="text-align:center; color:#1B8754; font-weight:bold;">Growth-factor co-receptors & signaling modulators</p>
:::
:::
::::
:::::

## [ToD candidates as Transcription factors]{.r-fit-text}{.smaller}

::: {layout-nrow=2}

:::{.fragment}
:::{.callout-note}
## [1] Identify Translation-on-Demand Genes

- Yang time-course: keep genes whose
- Protein LFC » RNA LFC
- RNA stays in "stable" window (-2 ≤ RNA LFC ≤ 2)

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">438 ToD genes</p>
:::
:::

:::{.fragment .smaller}
::: callout-caution
## [2] Isolate Transcription Factors

- Cross-check ToD list with
- AnimalTFDB v4
- TFs are significantly over-represented

<p style="text-align:center; color:#FE7E14; font-weight:bold;">46 TFs (10.5% of 438 ToD genes) → p = 0.021</p>
:::
:::

:::{.fragment .smaller}
::: callout-caution
## [3] Retain TFs with Mapped Targets

- TEs have curated regulatory networks in
- CollecTRI

<p style="text-align:center; color:#FE7E14; font-weight:bold;">22 TFs → taken forward</p>
:::
:::

:::{.fragment .smaller}
::: callout-tip
## [4] Infer TF Activity

* decoupleR (ULM) - Every TF-target link has a sign:
  + +1 = activator (targets T when TF active)
  + -1 = repressor (targets / when TF active)
* ULM regresses target-gene expression on signed weights:
  + Activator targets up OR repressor targets down → positive t-value
  + Opposite expectations → negative t-value
  

<p style="text-align:center; color:#1B8754; font-weight:bold;">t-value = TF activity score → heatmap</p>
:::
:::

:::



## [ToD candidates as Transcription factors]{.r-fit-text}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/TF_heatmap.svg)
::::
::::{.column width="40%"}

:::{.fragment .incremental}
::: callout-note
## Known important genes during the transition are ToD candidates

- **Hsf2**: Heat Shock Transcription Factor → protects against cellular stress
- **Foxd3**: Shuts naïve enhancers → kick-starts exit from naïve state
- **Klf10**: Member of KLF family → tilts balance toward differentiation
:::
:::

::::
:::::

## [How are these genes selectively post-transcriptionally upregulated?]{.r-fit-text}{.smaller}
::: {layout-nrow=2}

:::{.fragment}
:::{.callout-note .smaler}
## [1] UTR Collection

Take the 3'-UTR sequences of all Translation-on-Demand (ToD) candidate genes for each time-window.

<p style="text-align:center; color:#0B6FFD; font-weight:bold;">Target gene UTR sequences</p>
:::
:::

:::{.fragment .smaller}
::: callout-note
## [2] Background

Use the 3'-UTRs of all other genes detected in both transcriptome AND proteome as the control universe.

<p style="text-align:center; color:#FE7E14; font-weight:bold;">Control UTR background</p>
:::
:::

:::{.fragment .smaller}
::: callout-caution
## [3] Motif Scan (AME, MEME-suite)

Run AME with the CISBP-RNA motif library

- Parameters: Fisher's exact test, adj-p cut-off < 0.05

<p style="text-align:center; color:#FE7E14; font-weight:bold;">Enriched RBP motifs</p>
:::
:::

:::{.fragment .smaller}
::: callout-tip
## [4] Check Proteins

Cross-checked enriched RBPs in the Yang proteome to see whether they themselves change over
  

<p style="text-align:center; color:#1B8754; font-weight:bold;">RBP temporal dynamics</p>
:::
:::

:::

## [Are RBP motifs enriched in 3’ UTRs of ToD candidates?]{.r-fit-text}{.smaller}
::::: columns
::::{.column width="60%"}
![](figures/defense_specific/motif_plot_3_prime.svg)
::::
::::{.column width="40%"}

:::{.fragment .incremental}
::: callout-note
## RBPs important for early development

-  **Lin28**: interacts within large <span style="color:#d32f2f; font-weight:600;">ribonucleoprotein complexes</span> to modulate translation of key <span style="color:#d32f2f; font-weight:600;">developmental mRNAs</span>
-  **Pcbp1-3**: modulating <span style="color:#d32f2f; font-weight:600;">mRNA stability</span>, <span style="color:#d32f2f; font-weight:600;">alternative splicing</span> as well as translation regulation
-  **Ptbp1**: integrating <span style="color:#d32f2f; font-weight:600;">developmental cues</span> in translational regulation and shape post-transcriptional modification by regulating <span style="color:#d32f2f; font-weight:600;">alternative splicing</span>
:::
:::




::::
:::::



## The picture so far...

::: callout-tip
## Insights
 - ToD is a detectable post-trascriptional regulatory mechanism during early develompent
 - ToD aids correct transition from the naive to the formative state
 - The biological implications from the data agrees with the proposed purpose of ToD
:::

::: {.fragment}
::: callout-important
## Limitations
- We do not have strong biological signals… e.g. in the enrichments, motif enrichmets etc.
- Methods need to be adjusted for other datasets (scaling issues)
- Medium Change confounding stress
:::
:::


## Next steps? 

:::{.fragment fragment-index=1}
- Polysome profiling
:::
:::{.fragment fragment-index=2}
- Human dataset ToD calling
:::
:::{.fragment fragment-index=3}
- Other data normalization methods (GPR, LOESS) → better suitable for temporal data with small replicate number
:::


::::: columns
::::{.column width="33%"}
:::{.fragment fragment-index=1}
![](figures/defense_specific/Mono_poly.png){width="60%"}
:::
::::
::::{.column width="33%"}
:::{.fragment fragment-index=2}
![](figures/defense_specific/human.jpg){width="60%"}
:::
::::
::::{.column width="33%"}
:::{.fragment fragment-index=3}
![](figures/defense_specific/LR.png){width="60%"}
:::
::::
:::::





## Thank you so much, Beyer Crew! {.smaller background-color="white"}

::::: columns

::::{.column width="50%" .smaller}

Supervisor: 

special thanks to **Luke Brandwood**

![](figures/defense_specific/luke_backpack.png){width=60%}

Reviewers: 

a huge thanks goes to...

- **Prof. Dr. Andreas Beyer** 
- **Prof. Dr. Achim Tresch**

::::

::::{.column width="50%"}

::: {layout="[[1],[1,1]]"}
![](figures/defense_specific/Beyer_lab.jpg){width=90%} 

![](Uni_Köln_Logo.png){width=50%} 

![](CECAD_logo.jpg){width=50%}
:::

::::

:::::

## Datasets for Translation on Demand {.smaller}

| Dataset                          | Temporal resolution       | Replicates Transcriptomics | Replicates Proteomics | Organism |
|:---------------------------------|:--------------------------|:--------------------------|:----------------------|:---------|
| <span style="color: red;">Yang Data</span> | 0, 1, 6, 12, 24, 48, 72   | 2                          | 4                     | Mice     |
| Leeb TC Data (Time Course)       | 0, 4, 8, 12, 24, 32       | 1                          | 4                     | Mice     |
| Leeb (MC) Data Medium Change     | 0, 4, 8                   | 3                          | 4                     | Mice     |
| Leeb Human TC                    | 0, 1, 12, 48, 72, 96      | 2                          | 4                     | Human    |

- In the making: Polysome Profiling

## Testing out data smoothing methods

![](figures/defense_specific/comparison_of_smoothers.png)

## ToD candidates in other datasets
![](figures/early_tods_venn_diagram.svg)

## What processes are subject to ToD?
![](figures/ORA_plot.svg)

---

![](figures/sub_figures/GO_Reactome_Terms_subfig.svg)


---

![](figures/defense_specific/motif_plot.svg)
